<!DOCTYPE html>
<!--[if lt IE 7]> <html class="ie6 ie"> <![endif]-->
<!--[if IE 7]>    <html class="ie7 ie"> <![endif]-->
<!--[if IE 8]>    <html class="ie8 ie"> <![endif]-->
<!--[if IE 9]>    <html class="ie9 ie"> <![endif]-->
<!--[if !IE]> --> <html> <!-- <![endif]-->

<!-- Mirrored from seop.illc.uva.nl/entries/biology-developmental/ by HTTrack Website Copier/3.x [XR&CO'2014], Wed, 22 Jun 2022 19:39:43 GMT -->
<head>
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<title>
Developmental Biology (Stanford Encyclopedia of Philosophy)
</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="robots" content="noarchive, noodp" />
<meta property="citation_title" content="Developmental Biology" />
<meta property="citation_author" content="Love, Alan" />
<meta property="citation_publication_date" content="2015/03/23" />
<meta name="DC.title" content="Developmental Biology" />
<meta name="DC.creator" content="Love, Alan" />
<meta name="DCTERMS.issued" content="2015-03-23" />
<meta name="DCTERMS.modified" content="2020-02-14" />

<!-- NOTE: Import webfonts using this link: -->
<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,300,600,200&amp;subset=latin,latin-ext" rel="stylesheet" type="text/css" />

<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/bootstrap.min.css" />
<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/bootstrap-responsive.min.css" />
<link rel="stylesheet" type="text/css" href="../../css/font-awesome.min.css" />
<!--[if IE 7]> <link rel="stylesheet" type="text/css" href="../../css/font-awesome-ie7.min.css"> <![endif]-->
<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/style.css" />
<link rel="stylesheet" type="text/css" media="print" href="../../css/print.css" />
<link rel="stylesheet" type="text/css" href="../../css/entry.css" />
<!--[if IE]> <link rel="stylesheet" type="text/css" href="../../css/ie.css" /> <![endif]-->
<script type="text/javascript" src="../../js/jquery-1.9.1.min.js"></script>
<script type="text/javascript" src="../../js/bootstrap.min.js"></script>

<!-- NOTE: Javascript for sticky behavior needed on article and ToC pages -->
<script type="text/javascript" src="../../js/jquery-scrolltofixed-min.js"></script>
<script type="text/javascript" src="../../js/entry.js"></script>

<!-- SEP custom script -->
<script type="text/javascript" src="../../js/sep.js"></script>
</head>

<!-- NOTE: The nojs class is removed from the page if javascript is enabled. Otherwise, it drives the display when there is no javascript. -->
<body class="nojs article" id="pagetopright">
<div id="container">
<div id="header-wrapper">
  <div id="header">
    <div id="branding">
      <div id="site-logo"><a href="../../index.html"><img src="../../symbols/sep-man-red.png" alt="SEP home page" /></a></div>
      <div id="site-title"><a href="../../index.html">Stanford Encyclopedia of Philosophy</a></div>
    </div>
    <div id="navigation">
      <div class="navbar">
        <div class="navbar-inner">
          <div class="container">
            <button class="btn btn-navbar collapsed" data-target=".collapse-main-menu" data-toggle="collapse" type="button"> <i class="icon-reorder"></i> Menu </button>
            <div class="nav-collapse collapse-main-menu in collapse">
              <ul class="nav">
                <li class="dropdown open"><a id="drop1" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-book"></i> Browse</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop1">
                    <li><a href="../../contents.html">Table of Contents</a></li>
                    <li><a href="../../new.html">What's New</a></li>
                    <li><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/random">Random Entry</a></li>
                    <li><a href="../../published.html">Chronological</a></li>
                    <li><a href="../../archives/index.html">Archives</a></li>
                  </ul>
                </li>
                <li class="dropdown open"><a id="drop2" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-info-sign"></i> About</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop2">
                    <li><a href="../../info.html">Editorial Information</a></li>
                    <li><a href="../../about.html">About the SEP</a></li>
                    <li><a href="../../board.html">Editorial Board</a></li>
                    <li><a href="../../cite.html">How to Cite the SEP</a></li>
                    <li><a href="../../special-characters.html">Special Characters</a></li>
                    <li><a href="../../tools/index.html">Advanced Tools</a></li>
                    <li><a href="../../contact.html">Contact</a></li>
                  </ul>
                </li>
                <li class="dropdown open"><a id="drop3" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-leaf"></i> Support SEP</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop3">
                    <li><a href="../../support/index.html">Support the SEP</a></li>
                    <li><a href="../../support/friends.html">PDFs for SEP Friends</a></li>
                    <li><a href="../../support/donate.html">Make a Donation</a></li>
                    <li><a href="../../support/sepia.html">SEPIA for Libraries</a></li>
                  </ul>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
    <!-- End navigation -->
    
    <div id="search">
      <form id="search-form" method="get" action="https://seop.illc.uva.nl/search/searcher.py">
        <input type="search" name="query" placeholder="Search SEP" />
        <div class="search-btn-wrapper"><button class="btn search-btn" type="submit"><i class="icon-search"></i></button></div>
      </form>
    </div>
    <!-- End search --> 
    
  </div>
  <!-- End header --> 
</div>
<!-- End header wrapper -->

<div id="content">

<!-- Begin article sidebar -->
<div id="article-sidebar" class="sticky">
  <div class="navbar">
    <div class="navbar-inner">
      <div class="container">
        <button class="btn btn-navbar" data-target=".collapse-sidebar" data-toggle="collapse" type="button"> <i class="icon-reorder"></i> Entry Navigation </button>
        <div id="article-nav" class="nav-collapse collapse-sidebar in collapse">
          <ul class="nav">
            <li><a href="#toc">Entry Contents</a></li>
            <li><a href="#Bib">Bibliography</a></li>
            <li><a href="#Aca">Academic Tools</a></li>
            <li><a href="https://leibniz.stanford.edu/friends/preview/biology-developmental/">Friends PDF Preview <i class="icon-external-link"></i></a></li>
            <li><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/archinfo.cgi?entry=biology-developmental">Author and Citation Info <i class="icon-external-link"></i></a> </li>
            <li><a href="#pagetopright" class="back-to-top">Back to Top <i class="icon-angle-up icon2x"></i></a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
<!-- End article sidebar --> 

<!-- NOTE: Article content must have two wrapper divs: id="article" and id="article-content" -->
<div id="article">
<div id="article-content">

<!-- BEGIN ARTICLE HTML -->


<div id="aueditable"><!--DO NOT MODIFY THIS LINE AND ABOVE-->

<h1>Developmental Biology</h1><div id="pubinfo"><em>First published Mon Mar 23, 2015; substantive revision Fri Feb 14, 2020</em></div>

<div id="preamble">

<p>
Developmental biology is the science that investigates how a variety
of interacting processes generate an organism&rsquo;s heterogeneous
shapes, size, and structural features that arise on the trajectory
from embryo to adult, or more generally throughout a life cycle. It
represents an exemplary area of contemporary experimental biology that
focuses on phenomena that have puzzled natural philosophers and
scientists for more than two millennia. Philosophers of biology have
shown interest in developmental biology due to the potential relevance
of development for understanding evolution, the theme of reductionism
in genetic explanations, and via increased attention to the details of
particular research programs, such as stem cell biology. Developmental
biology displays a rich array of material and conceptual practices
that can be analyzed to better understand the scientific reasoning
exhibited in experimental life science. This entry briefly reviews
some central phenomena of ontogeny and then explores four domains that
represent some of the import and promise of conceptual reflection on
the epistemology of developmental biology.</p>
</div>

<div id="toc">
<!--Entry Contents-->
<ul>
 <li><a href="#Ove">1. Overview</a>
<ul>
 <li><a href="#HisCon">1.1 Historical Considerations</a></li>
 <li><a href="#DevPhe">1.2 Developmental Phenomena</a></li>
 <li><a href="#DevMech">1.3 Developmental Mechanisms</a></li>
 </ul></li>
 <li><a href="#EpiOrgDevBio">2. The Epistemological Organization of Developmental Biology</a>
<ul>
 <li><a href="#NoTheDev">2.1 No Theory of Development?</a></li>
 <li><a href="#EroOrg">2.2 Erotetic Organization</a></li>
 </ul></li>
 <li><a href="#ExpAppDev">3. Explanatory Approaches to Development</a>
<ul>
 <li><a href="#Gen">3.1 Genetics</a></li>
 <li><a href="#Phy">3.2 Physics</a></li>
 <li><a href="#IntAppGenPhy">3.3 Integrating Approaches: Genetics and Physics</a></li>
 </ul></li>
 <li><a href="#ModOrgForStuDev">4. Model Organisms for the Study of Development</a></li>
 <li><a href="#DevEvo">5. Development and Evolution</a>
<ul>
 <li><a href="#FunHomDevGen">5.1 Functional Homology in Developmental Genetics</a></li>
 <li><a href="#NorStaPhePla">5.2 Normal Stages and Phenotypic Plasticity</a></li>
 </ul></li>
 <li><a href="#Con">6. Conclusion</a></li>
 <li><a href="#Bib">Bibliography</a>
<ul>
 <li><a href="#Ref">References</a></li>
 <li><a href="#FigCre">Figure Credits</a></li>
 </ul></li>
 <li><a href="#Aca">Academic Tools</a></li>
 <li><a href="#Oth">Other Internet Resources</a></li>
 <li><a href="#Rel">Related Entries</a></li>
 </ul>
<!--Entry Contents-->
<hr />
</div>

<div id="main-text">

<h2><a id="Ove">1. Overview</a></h2>

<h3><a id="HisCon">1.1 Historical Considerations</a></h3>

<p>
Developmental biology is the science that investigates how a variety
of interacting processes generate an organism&rsquo;s heterogeneous
shapes, size, and structural features that arise on the trajectory
from embryo to adult, or more generally throughout a life cycle (Love
2008; Minelli 2011a). It represents an exemplary area of contemporary
experimental biology that focuses on phenomena that have puzzled
natural philosophers and scientists for more than two millennia. How
do the dynamic relations among seemingly homogeneous components in the
early stages of an embryo produce a unified whole organism containing
heterogeneous parts in the appropriate arrangement and with correct
interconnections? More succinctly, how do we explain <em>ontogeny</em>
(or, more archaically, <em>generation</em>)? In <em>Generation of
Animals</em>, Aristotle provided the first systematic investigation of
developmental phenomena and recognized key issues about the emergence
of and relationships between hierarchically organized parts (e.g.,
bone and anatomical features containing bone), as well as the
explanatory difficulty of determining how a morphological form is
achieved reliably in offspring (e.g., the typical shape and structure
of appendages in a particular species). Generation remained a poignant
question throughout the early modern period and was explored by many
key figures writing at the time, including William Harvey, Ren&eacute;
Descartes, Robert Boyle, Pierre Gassendi, Nicolas Malebranche,
Gottfried Wilhelm Leibniz, Anne Conway, Immanuel Kant, and others
(Smith 2006). Observations of life cycle transitions, such as
metamorphosis, fed into these endeavors and led to striking
conclusions, such as Leibniz&rsquo;s denial of generation <em>sensu
stricto</em>.</p>

<blockquote>

<p>
Animals and all other organized substances have no
beginning &hellip; their apparent generation is only a development, a
kind of augmentation &hellip; a transformation like any other, for
instance like that of a caterpillar into a butterfly. (Smith 2011:
186&ndash;187)</p>
</blockquote>

<p>
A major theme that crystallized in this history of investigation is
the distinction between epigenesis and preformation (see the entry on
 <a href="../theories-biological-development/index.html">theories of biological development</a>).
 Proponents of epigenesis claimed that heterogeneous, complex features
of form emerge from homogeneous, less complex embryonic structures
through interactive processes. Thus, an explanation of the ontogeny of
these form features requires accounting for how the interactions
occur. Proponents of preformation claimed that complex form preexists
in the embryo and &ldquo;unfolds&rdquo; via ordinary growth processes.
An adequate explanation involves detailing how growth occurs. Although
preformation has a lighter explanatory burden in accounting for how
form emerges during ontogeny (on the assumption that growth is easier
to explain than process interactions), it also must address how the
starting point of the next generation is formed with the requisite
heterogeneous complex features. This was sometimes accomplished by
embedding smaller and smaller miniatures <em>ad infinitum</em> inside
the organism
 (<a href="#fig1">Figure 1</a>).
 Epigenetic perspectives were often dependent on forms of teleological
reasoning (see the entry on
 <a href="../teleology-biology/index.html">teleological notions in biology</a>)
 to account for why interactions among homogeneous components
eventually result in a complex, integrated whole organism. Though
nothing prevents mixing features of these two outlooks in explaining
different aspects of development, polarization into dichotomous
positions has occurred frequently (Rose 1981; Smith 2006).</p>

<div id="fig1" class="figure">
<img src="Figure1.gif" style="width:200px;" alt="[three drawings, the first is like a balloon with a scrunched up human inside and a string hanging down, the second of a dangling human figure and the third of a somewhat larger dangling human figure.]" />

<p>
<span class="figlabel">Figure 1:</span> An early modern depiction of a
tiny person inside of a sperm exemplifying preformationist views. </p>
</div>

<p>
In the late 19<sup>th</sup> and early 20<sup>th</sup> century, the
topic of development was salient in controversies surrounding
vitalism, such as the disagreement between Wilhelm Roux and Hans
Driesch over how to explain ontogeny (Maienschein 1991). Roux thought
that a fertilized egg contains inherited elements that represent
different organismal characteristics. During the process of cellular
division, these elements become unequally distributed among daughter
cells leading to distinct cell fates. Driesch, in contrast, held that
each cell retained its full potential through division even though
differentiation occurred. Although this issue is often understood in
terms of the metaphysics of life (vitalism <em>versus</em>
materialism), Driesch&rsquo;s interpretation of development and the
autonomy of an organism had epistemological dimensions (Maienschein
2000). The explanatory disagreement involved different experimental
approaches and divergent views on the nature of differentiation in
early ontogeny (e.g., to what degree cells are pre-specified). A
familiar philosophical theme running through these discussions, both
epistemological and metaphysical, is the status of
 <a href="../reduction-biology/index.html">reductionism in biology</a>.
 Through the middle of the 20<sup>th</sup> century,
embryology&mdash;the scientific discipline studying
development&mdash;slowly transformed into developmental biology with a
variety of reworked and recalcitrant elements (Berrill 1961). In
conjunction with the issue of reductionism, a key aspect of this
history is the molecularization of experimental (as opposed to
comparative) embryology (Fraser and Harland 2000), with a concomitant
emphasis on the explanatory power of genes (see the entry on
 <a href="../gene/index.html">gene</a>
 and
 <a href="#Gen">Section 3.1</a>).
 This complex and fascinating history, including interrelations with
medicine and reproductive technology, has been detailed elsewhere
(see, e.g., Oppenheimer 1967; Horder et al. 1986; Hamburger 1988;
Hopwood 2019; Maienschein 2014; Maienschein et al. 2005; Gilbert 1991;
Embryo Project in Other Internet Resources).</p>

<p>
Developmental biology has increasingly become an area of exploration
for
 <a href="../biology-philosophy/index.html">philosophy of biology</a>
 due to the potential relevance of development for understanding
evolution (Love 2015;
 <a href="#DevEvo">Section 5</a>),
 the theme of reductionism in biology and explanations from molecular
genetics (Robert 2004; Rosenberg 2006;
 <a href="#ExpAppDev">Section 3</a>),
 and via increased attention to the details of particular research
programs, such as stem cell biology (Fagan 2013; Laplane 2016).
However, it should not be forgotten that ontogeny was on the radar of
philosophical scholars in the 20<sup>th</sup> century, as seen in
Ernest Nagel&rsquo;s treatment of hierarchical organization and
reduction in the development of living systems (Nagel 1961: 432ff).
For contemporary philosophy of science, developmental biology displays
a rich array of material and conceptual practices that can be analyzed
to better understand the scientific reasoning exhibited in
experimental life science (see the entry on
 <a href="../biology-experiment/index.html">experiment in biology</a>).
 After a brief review of some central phenomena of ontogeny, this
entry explores four domains that represent some of the import and
promise of conceptual reflection on the epistemology of developmental
biology.</p>

<h3><a id="DevPhe">1.2 Developmental Phenomena</a></h3>

<blockquote>

<p>
Developmental biology is the science that seeks to explain how the
structure of organisms changes with time. Structure, which may also be
called morphology or anatomy, encompasses the arrangement of parts,
the number of parts, and the different types of parts. (Slack 2006:
6)</p>
</blockquote>

<p>
Most of the properties that developmental biologists attempt to
explain are structural rather than functional. For example, a
developmental biologist concentrates more on how tissue layers fold or
how shape is generated than on what the folded tissue layers do or how
the shape functions. The ontogeny of function, at all levels of
organization, is an element of developmental biology, but it is often
bracketed because of the predominance (both past and present) of
questions surrounding the ontogeny of form or structure (Love
2008).</p>

<p>
Textbooks (e.g., Gilbert 2010; Slack 2013; Wolpert et al. 2010)
typically describe a canonical set of events surrounding the changing
structures displayed during animal
 development.<sup>[<a href="notes.html#note-1" id="ref-1">1</a>]</sup>
 The first of these is <em>fertilization</em> (in sexually reproducing
species), where an already semi-organized egg merges with a sperm
cell, followed by the fusion of their nuclei to achieve the
appropriate complement of genetic material. Second, the fertilized egg
undergoes several rounds of <em>cleavage</em>, which are mitotic
divisions without cell growth that subdivide the zygote into many
distinct cells
 (<a href="#fig2">Figure 2</a>).
 After many rounds of cleavage, this spherical conglomerate of cells
(now called a <em>blastula</em>) begins to exhibit some specification
of <em>germ layers</em> (endoderm, mesoderm, and ectoderm) and then
proceeds to invaginate at one end, a complex process referred to as
<em>gastrulation</em> that eventually yields a through-gut. All three
germ layers, from which specific types of cells are derived (e.g.,
neural cells from ectoderm), become established during gastrulation or
shortly after it
 completes.<sup>[<a href="notes.html#note-2" id="ref-2">2</a>]</sup>
 <em>Organogenesis</em> refers to the production of tissues and organs
through the interaction and rearrangement of cell groups. Events
confined to distinct taxonomic groups include <em>neurulation</em> in
chordates, whereas other events correlate with mode of development
(<em>metamorphosis</em> from a larval to adult stage) or individual
trauma (<em>regeneration</em> of a limb).</p>

<div id="fig2" class="figure">
<img src="Figure2.png" alt="[6 columns by 2 rows. The second row is a sideways view of the item in the first row. The first column (A) has four orange colored cells labelled 'A', 'B', 'C', and 'D'. The second column (B) has 8 cells, the bigger lower orange ones labelled '1A', '1B', '1C', and '1D'; the smaller pink ones on top labelled '1a', '1b', '1c', and '1d'. The third column (C) has 12 cells with the bottom, orange, labelled '2A' through '2D', the middle, red, labelled '2a' through '2d' and the top, pink, labelled '1a' through '1d'. The fourth column (D) has 16 cells, bottom layer, orange, is '2A' through '2D', top, pink, is '1a1' through '1d1', the middle alternates between larger red cells labelled '2a' through '2d' and smaller beige cells labelled '1a2' through '1d2'. The fifth column (E) has at the bottom orange cells labelled '3A' through '3D', at the top a cross of four pink cells labelled '1a1' through '1d1' and between the branches of the cross four beige cells labelled '1a2' through '1d2'. In the middle are large red cells labelled '2a' through '2d' and squished between the orange, beige, and red cells are dark red cells labelled '3a' through '3d'. The last column (F) is different. On the top is a clump of cells colored blue, green, pink, and orange. On the bottom on the left side is an arrow pointed up labelled 'developmental time'. Next to it is a 6 rowed drawing. The first row on the bottom extends the full width of the column and contains the word 'zygote', the row above is split in two, on the left 'AB' and on the left 'CD'. The third row is split evenly in 4: green 'A', blue 'B', orange 'C', and red 'D'. The fourth row is split evenly into 8 each shaded the same color as the unit below (though with different intensities). The fifth row is split evenly into 16 and the sixth row into 32 following the same pattern as the fourth row.]" />

<p>
<span class="figlabel">Figure 2:</span> An example of embryonic
cleavage in marine snail embryos showing the fate of different cell
lineages through developmental time.</p>
</div>

<p>
Several key processes underlie these distinct developmental events and
the resulting features of form that emerge (e.g., the through-gut
formed subsequent to gastrulation or the heart formed during
organogenesis). These are critical to the ontogeny of form and link
directly to major research questions in developmental biology
 (<a href="#EpiOrgDevBio">Section 2</a>).
 First, cellular properties, such as shape, change during ontogeny.
This is a function of <em>differentiation</em> whereby cells adopt
specific fates that include shape transformations
 (<a href="#fig3">Figure 3</a>).
 Second, regions of cells in the embryo are designated through
arrangement and composition alterations that correspond to different
axes in different parts of the embryo (e.g., dorsal-ventral,
anterior-posterior, left-right, and proximal-distal). The successive
establishment of these regions is referred to as <em>pattern
formation</em>. Third, cells translocate and aggregate into layers
(e.g., endoderm and ectoderm, followed by the mesoderm in many
lineages) and later tissues (aggregations of differentiated cell
types). Fourth, cells and tissues migrate and interact to produce new
arrangements and shapes composed of multiple tissue layers with novel
functions (i.e., organs). These last two sets of processes are usually
termed <em>morphogenesis</em> (Davies 2005) and occur via many
distinct mechanisms
 (<a href="#DevMech">Section 1.3</a>).
 Fifth, there is <em>growth</em> in the size of different form
features in the individual, remarkably obvious when comparing zygote
to adult, although proportional change between different features
(<em>allometry</em>) is also striking.</p>

<div id="fig3" class="figure">

<!--pdf include <img src="Figure3.png" style="width:100%;"/> pdf include-->

<!--pdf exclude begin-->
<img src="Figure3.svg" style="width:100%;" alt="[A tree graph of 5 levels. The top node 'Multipotential hematopoietic stem cell (Hemocytoblast)' points back to itself as well as pointing to the next level. On the left 'Common myeloid progenitor' which points to 'Megakarycocyte' (which in turn points to 'Thrombocytes'), 'Erythrocyte', 'Mast cell', and 'Myeloblast' (which in turn points to 'Basophil', 'Neutrophil', 'Eosinophil', 'Monocyte' (which in turn points to 'Macrophage')). On the right 'Common lymphoid progenitor' which points to 'Natural killer cell (Large granual lymphocyte)' and 'Small lymphocyte' (which in turn points to 'T lymphocyte' and 'B lymphocyte' (which in turn points to 'Plasma cell')).]" />
<!--pdf exclude end-->

<p>
<span class="figlabel">Figure 3:</span> A simple illustration of the
kinds of differentiation related to the cellular components found in
blood.</p>
</div>

<p>
None of these processes occur in isolation and explanations of
particular form features usually draw on several of them
simultaneously, presuming other features that originated earlier in
ontogeny by different instantiations and combinations of the
processes. This sets a broad agenda for investigation: how do various
iterations and combinations of these processes generate form features
during ontogeny? Consider the concrete example of vertebrate
cardiogenesis. How does the vertebrate heart, with its internal and
external shape and structure originate during ontogeny (Harvey 2002)?
How does the heart come to exhibit left/right asymmetry in the body
cavity? What causes cells to adopt a muscle cell fate or certain
tissues to interact in the prospective region of the heart? How do
muscle cells migrate to, aggregate in, and differentiate at the
correct location? How does the interior of the heart adopt a
particular tubular structure with various chambers (that differs among
vertebrate species)? How does the heart grow at a particular rate and
achieve a specific size? Solutions relevant to explaining the ontogeny
of form characterize causal factors that account for how different
processes occur and yield various outcomes
 (<a href="#ExpAppDev">Section 3</a>).</p>
 
<h3><a id="DevMech">1.3 Developmental Mechanisms</a></h3>

<p>
A developmental <i>mechanism</i> is a mechanism or process that
operates during ontogeny (see McManus 2012 for discussion). At least
two different types of developmental mechanisms can be distinguished
(Love 2017a): molecular genetic mechanisms (signaling or gene
regulatory networks;
 <a href="#Gen">Section 3.1</a>)
 and cellular-physical mechanisms (cell migration or epithelial
invagination;
 <a href="#Phy">Section 3.2</a>).
 Philosophical explorations of mechanisms in science and mechanistic
explanation have grown dramatically over the past two decades (Craver
and Darden 2013; Glennan and Illari 2017; Illari and Williamson 2012).
Among different accounts of scientific mechanisms, four shared
elements are discernable: (1) what a mechanism is for, (2) its
constituents, (3) its organization, and, (4) the spatiotemporal
context of its operation. Developmental explanations seek to
characterize these four elements through various experimental
interventions. Together these elements provide a template for
characterizing the two different types of developmental
mechanisms.</p>

<p>
A well-established molecular genetic mechanism is the initial
formation of segments in <i>Drosophila</i> due to the segment polarity
network of gene expression (Wolpert et al. 2010, 70-81; Damen 2007).
By Stage 8 of development (~3 hours post-fertilization),
<i>Drosophila</i> embryos have 14 parasegment units that were defined
by pair-rule gene expression in earlier stages. The transcription
factor Engrailed accumulates in the anterior portion of each
parasegment. This initiates a cascade of gene activity that defines
the boundaries of each compartment of cells that will eventually
become a segment. One element of this activity is the expression of
<i>hedgehog</i>, a secreted signaling protein, in cells anterior to
the band of cells where Engrailed has accumulated, which marks the
posterior boundary of each nascent segment. This, in turn, activates
the expression of <i>wingless</i>, another secreted signaling protein,
which maintains the expression of both <i>engrailed</i> and
<i>hedgehog</i> in a feedback loop so that segment boundaries persist
 (<a href="#fig4">Figure 4</a>).
 The segment polarity network exhibits all four of the shared elements
of a mechanism. It is constituted by a number of parts (e.g.,
Engrailed, Wingless, Hedgehog) and activities or component operations
(e.g., signaling proteins bind receptors, transcription factors bind
to DNA and initiate gene expression), which are organized into
patterns of interacting relationships (feedback loops, signaling
pathways) within a spatiotemporal context (in parasegments of the
<i>Drosophila</i> embryo, ~3 hours post-fertilization) so as to
produce a specific behavior or phenomenon (a set of distinct segments
with well-defined boundaries).</p>

<div id="fig4" class="figure">
<img src="Figure4.png" style="width:200px;" alt="Wingless and Hedgehog reciprocal signaling during segmentation of Drosophila embryos" />

<p>
<span class="figlabel">Figure 4:</span> Wingless and Hedgehog
reciprocal signaling during segmentation of <em>Drosophila</em>
embryos.</p>
</div>

<p>
Next, consider the cellular-physical mechanism of branching
morphogenesis, which refers to combinations of cellular proliferation
and movement that yield branch-like structures in kidneys, lungs,
glands, or blood vessels. There are many types of branching
morphogenesis, but one primary mechanism is epithelial folding, which
involves cells invaginating at different locations on a structure to
yield branches (Davies 2013, ch. 20). Different cellular-physical
mechanisms can produce invaginations that lead to branching structures
(Varner and Nelson 2014): the constriction of one end of a subset of
columnar cells in an epithelium (&ldquo;apical constriction&rdquo;);
increased cell proliferation of one epithelial sheet in relation to
another (&ldquo;differential growth&rdquo;); and compression of an
epithelium leading to periodic invaginations (&ldquo;mechanical
buckling&rdquo;). That different mechanisms can lead to the same
morphological outcome means it can be difficult to discern which
mechanism is operating in an embryonic context. Branching
morphogenesis also exhibits all four of the shared elements of a
mechanism. The parts are cells and tissues with activities or
component operations (e.g., apical constriction, differential growth,
mechanical buckling) being organized into patterns of interacting
relationships (apical constriction leading to epithelial invagination)
within a spatiotemporal context (in tracheal precursors within the
<i>Drosophila</i> embryo around Stage 7 and 8). This organization
produces a specific behavior or phenomenon (a set of branching
structures&mdash;the trachea).</p>

<p>
Once these types of developmental mechanisms have been distinguished,
several conceptual issues become salient. The first pertains to how
the two types of mechanisms are interrelated during ontogeny, and how
different investigative approaches do or do not successfully provide
integrated accounts of them
 (<a href="#IntAppGenPhy">Section 3.3</a>).
 A second is their distinct patterns of generality. Molecular genetic
mechanisms are widely conserved across phylogenetically disparate taxa
as a consequence of evolutionary descent, whereas cellular-physical
mechanisms are widely instantiated as a consequence of shared physical
organization but not due to evolutionary descent (Love 2017a). The
divergence of these patterns has prompted explicit epistemological
reflection by developmental
 biologists.<sup>[<a href="notes.html#note-3" id="ref-3">3</a>]</sup></p>
 
<h2><a id="EpiOrgDevBio">2. The Epistemological Organization of Developmental Biology</a></h2>

<p>
One recurring theme in the long history of investigations into
development is that explaining the ontogeny of form consists of many
interrelated questions about diverse phenomena
 (<a href="#DevPhe">Section 1.2</a>).
 Sometimes philosophers have attempted to compress these questions
into one broad problem.</p>

<blockquote>

<p>
The real question concerning metazoan ontogeny is just how a single
cell gives rise to the requisite number of differentiated cell
lineages with all the right inductive developmental interactions
required to reproduce the form of the mature organism (Moss 2002:
97).</p>

<p>
The central problem of developmental biology is to understand how a
relatively simple and homogeneous cellular mass can differentiate into
a relatively complex and heterogeneous organism closely resembling its
progenitor(s) in relevant aspects (Robert 2004: 1).</p>
</blockquote>

<p>
This language is not necessarily incorrect but can lead to skewed
interpretations. For example, Philip Kitcher has argued that: </p>

<blockquote>

<p>
In contemporary developmental biology, there is &hellip; uncertainty
about how to focus the big, vague question, How do organisms develop?
(Kitcher 1993: 115)</p>
</blockquote>

<p>
This is simply false. While it is true that these questions have been
manifested with differing frequency and vigor through history, and the
ability to answer them (as well as the nature of the questions
themselves) has been contingent on different research strategies and
methods, the issue has not been an unwieldy central problem. But
scrutinizing the structure of developmental biology&rsquo;s questions
is not merely an exercise in clarification. It is crucial for
understanding how the science of developmental biology is
organized.</p>

<h3><a id="NoTheDev">2.1 No Theory of Development?</a></h3>

<p>
Although it is common in philosophy to associate sciences with
theories, such that the individuation of a science is dependent on a
constitutive theory or group of models, it is uncommon to find
presentations of developmental biology that make reference to a
<em>theory</em> of development (see discussion in Minelli and Pradeu
2014). Instead, we find references to families of approaches
(developmental genetics, experimental embryology, cell biology, and
 <a href="../molecular-biology/index.html">molecular biology</a>)
 or catalogues of &ldquo;key molecular components&rdquo;
(transcription factor families, inducing factor families, cytoskeleton
or cell adhesion molecules, and extracellular matrix components). No
standard theory or group of models provides theoretical scaffolding in
the major textbooks (e.g., Slack 2013; Wolpert et al. 2010; Gilbert
2010). The absence of any reference to a constitutive theory of
development or some set of core explanatory models is <em>prima
facie</em> puzzling. Three interpretations of this situation are
possible: (a) despite the lack of reference to theories, one can
reconstruct a theory (or theories) of developmental biology out of the
relevant discourse (e.g., multiple allied molecular models); (b) the
lack of reference to theories indicates an immaturity in developmental
biology because mature sciences always have systematic theories; and,
(c) the lack of reference to theories should be taken at face
value.</p>

<p>
Developmental biology is not an immature science, groping about for
some way to explain its phenomena: &ldquo;some of the basic processes
and mechanisms of embryonic development are now quite well
understood&rdquo; (Slack 2013: 7). The impetus for this type of
interpretation arises out of commitments to a conception of mature
science that presumes theories are abstract systems with a small set
of laws or core principles (see the entry on
 <a href="../structure-scientific-theories/index.html">the structure of scientific theories</a>).
 On the other hand, holding that developmental biology already has a
theory costumed in different guise&mdash;not referred to as such by
developmental biologists&mdash;is a possible interpretation. It arises
out of a view that sciences must have theories, which has been
expanded to allow for different understandings of theory structure,
such as constellations of models without laws, even though the
assumption is that theory still plays a similar organizing role in
guiding research. However, this assumption should be challenged and
rejected on methodological grounds in the case of developmental
biology. An analysis of the reasoning in a science should exhibit
epistemic transparency and not postulate &ldquo;hidden&rdquo;
reasoning structure (Love 2012). This criterion is based on the
premise that the basis of successes in scientific inquiry must be
available to those engaged in its practice (i.e., scientists). If we
postulate hidden structure not present in scientific discourse to
account for inductive inference, explanation, or other forms of
reasoning, then we risk obscuring how scientists themselves access
this structure to evaluate it (Woodward 2003: ch. 4). The successes of
developmental biology would become mysterious when viewed from the
vantage point of its participants.</p>

<p>
Epistemic transparency demands a descriptive correspondence between
philosophical accounts of science and scientific practice. This does
not mean that every claim made by any scientist should be taken with
the same credence. A ruling concern is pervasive features of practice.
The problem with assuming laws are required for explanation is their
relative absence from a variety of successful sciences routinely
offering explanations, not that no scientist ever appeals to laws as
explanatory. Pervasive features of scientific practice should be
prominent in philosophical accounts of sciences. Thus, it is not
surprising that the desire for a theory can be found among some
developmental biologists: &ldquo;Developing a theory is of utmost
importance for any discipline&rdquo; (Sommer 2009: 417). But the fact
that these calls are rare means we should not assume theories are
actually needed to govern and organize inquiry within the
 domain.<sup>[<a href="notes.html#note-4" id="ref-4">4</a>]</sup></p>
 
<p>
It was once thought that each science must have laws in order to offer
explanations (see the entry on
 <a href="../scientific-explanation/index.html">scientific explanations</a>),
 but now this is seen as unnecessary (Giere 1999; Woodward 2003). The
expectation that a science have a theory to accomplish the task of
organizing and guiding inquiry is of similar vintage. It derives from
an intuitive expectation of what counts as a mature science in the
first place. Even if we find empirically successful and coherent
traditions of research without a systematic theoretical framework
providing guidance, then the science cannot be mature. One might shrug
off these quasi-positivist appeals to maturity by invoking more
flexible conceptions of theory and theory structure. But why retain
the expectation that theories should accomplish the same epistemic
tasks? It is a preconception about knowledge structure that is not
plausible in light of the diversity of research practices found across
the sciences. The few scientists who favor this philosophical response
have different motivations. Instead of maturity, other reasons are
salient, such as guidance in the face of a welter of biochemical
detail or the need to forge a synthesis between evolution and
 development.<sup>[<a href="notes.html#note-5" id="ref-5">5</a>]</sup></p>
 
<p>
Developmental biologists recognize that the &ldquo;curse of
detail&rdquo; is one of the costs of developmental biology&rsquo;s
meteoric success over the past three decades: &ldquo;The principal
challenge today is that of exponentially increasing detail&rdquo;
(Slack 2013: ix). While something must provide organization and
guidance to developmental biology, it need not be theories that
accomplish the task. Regarding calls for a synthesis of evolution and
development, these often assume that having a developmental theory is
a precondition for synthesis (Sommer 2009): &ldquo;Our troubles
&hellip; derive from our standing lack of an explicit theory of
development&rdquo; (Minelli 2011a: 4). However, this line of argument
relies on the degree to which evolutionary theory exhibits the
supposed structure to which developmental biologists should aspire.
The actual practice associated with evolutionary theory indicates a
more flexible framework with chameleon qualities that is responsively
adjusted to the diverse investigative aims of evolutionary researchers
(Love 2013). Therefore, it is not clear that evolutionary theory
supplies the preferred template. A productive way forward is to
relinquish the prior expectation that sciences must have theories of a
certain kind to govern and guide their activity. Instead, sciences
that display empirical success and fecundity should be studied to
discover what features are responsible, without assuming that those
features will be the same for all sciences: &ldquo;Science need not be
understood in these terms and, indeed, may be better understood in
other terms&rdquo; (Giere 1999: 4).</p>

<h3><a id="EroOrg">2.2 Erotetic Organization</a></h3>

<p>
The criterion of epistemic transparency
 (<a href="#NoTheDev">Section 2.1</a>)
 encourages an exploration of our third interpretive option&mdash;the
lack of reference to theories should be taken at face value.
Developmental biology is organized primarily by stable, broad domains
of problems that correspond to abstract representations of major
ontogenetic processes (differentiation, pattern formation, growth, and
morphogenesis;
 <a href="#DevPhe">Section 1.2</a>).
 Yet how do we interpret the &ldquo;theoretical&rdquo; aspects of
developmental biology (e.g., positional information models of pattern
formation) and the utilization of theories from other domains (e.g.,
biochemistry)? One way is to distinguish between theory-informed
science&mdash;using theoretical knowledge&mdash;and theory-directed
science&mdash;having a theory that directs inquiry and organizes
knowledge (Waters 2007b); developmental biology is theory-informed but
not theory-directed. Theories need not be wholly absent from
developmental biology but&mdash;when present&mdash;they play roles
very different from standard philosophical expectations. Developmental
biology uses theoretical knowledge from biochemistry when appealing to
morphogen gradients to explain how segments are established or
chemical thermodynamics when invoking reaction&ndash;diffusion
mechanisms to explain pigmentation patterns. It also uses theoretical
knowledge derived from within developmental biology, such as
positional information models. Different kinds of theory inform
developmental biology, but these do not organize research&mdash;they
are not necessary to structure the knowledge and direct investigative
activities. Developmental biologists are not focused on confirming and
extending the theory of reaction&ndash;diffusion mechanisms, nor are
they typically organizing their research around positional
 information.<sup>[<a href="notes.html#note-6" id="ref-6">6</a>]</sup>
 This theoretical knowledge is used in building explanations but does
not provide rails of guidance for how to proceed in a research
program. All sciences may use theoretical knowledge, but this is not
the same as all sciences having a theory providing direction and
organization.</p>

<p>
Why think that problems provide organizational architecture for the
epistemology of developmental biology? They are a pervasive feature of
its reasoning practices, illustrated in textbooks that capture
substantial community consensus about standards of explanation,
experimental methods, essential concepts, and empirical content.
Unlike evolutionary biology textbooks that discuss the theory of
natural selection or economics textbooks that talk about microeconomic
theory, an examination of several editions of major textbooks indicate
that developmental biology does not have similar kinds of
theories.</p>

<p>
Jonathan Slack&rsquo;s <em>Essential Developmental Biology</em> (Slack
2006, 2013) is organized around four main types of processes, also
described as clustered groups of problems, which occur during
embryonic development: regional specification (pattern formation),
cell differentiation, morphogenesis, and growth. These broad clusters
are then fleshed out along a standard timeline of early development,
highlighting gametogenesis, fertilization, cleavage, gastrulation, and
axis specification
 (<a href="#DevPhe">see Section 1.2</a>).
 Different experimental approaches (cell and molecular biology,
developmental genetics, and experimental embryology) are utilized in a
specific set of model organisms
 (<a href="#ModOrgForStuDev">see below, Section 4</a>)
 to dissect the workings of these developmental phenomena. Subsequent
chapters cover later aspects of development (e.g., organogenesis),
with different systems treated in depth by tissue layer,
differentiation and growth, or in relation to evolutionary questions
(see below,
 <a href="#DevEvo">Section 5</a>).
 Throughout this presentation, no specific theory, set of hypotheses,
or dominant model is invoked to organize these different domains of
investigation. Instead, broad clusters of questions that reflect
generally delineated processes (differentiation, specification,
morphogenesis, and growth) set the agenda of research.</p>

<p>
Scott Gilbert&rsquo;s <em>Developmental Biology</em> exhibits a
similar pattern (Gilbert 2000 [2003, 2006, 2010]). Developmental
biology is constituted by two broad questions (&ldquo;How does the
fertilized egg give rise to the adult body? And how does that adult
body produce yet another body?&rdquo;), which can then be subdivided
into further categories, such as differentiation, morphogenesis,
growth, reproduction, regeneration, evolution, and environmental
regulation. These questions can be parsed more analytically in terms
of five variables: abstraction, variety, connectivity, temporality,
and spatial composition. The values given to these variables structure
the constellation of research questions within the broad problem
agendas corresponding to generally delineated processes. For example,
research questions oriented around events in zebrafish gastrulation
are structured in a way that differs from the research questions
oriented around vertebrate neural crest cell migration because they
involve different values for the five variables: abstraction
(zebrafish vs. vertebrates), temporality (earlier vs. later), spatial
composition (tissue layer interactions vs. a distinctive population of
cells), variety (epiboly vs. epithelium to mesenchyme transition), and
connectivity (gut formation and endoderm vs. organogenesis and
ectoderm/mesoderm). These configurations can be adjusted readily in
response to shifts in the values for different variables (Love
 2014).<sup>[<a href="notes.html#note-7" id="ref-7">7</a>]</sup></p>
 
<p>
This anatomy of problems, with explicit epistemological structure
derived from different values for these variables, operates to
organize the science of development. Investigators from different
disciplines can be working on the same problem but asking different
questions that require distinct but complementary methodological
resources. Knowledge and inquiry in developmental biology are
intricately organized, just not by a central theory or group of
models, and this erotetic organization is epistemologically accessible
to the participating scientists. While theoretical knowledge,
especially that drawn from molecular biological mechanisms (see the
entry on
 <a href="../molecular-biology/index.html">molecular biology</a>)
 and mathematical models (e.g., reaction&ndash;diffusion models) is
ubiquitous (<em>theory-informed</em>), the clusters of problems that
reappear across the textbooks and correspond to different types of
processes provide the governing architecture (<em>not
theory-directed</em>), which can be characterized explicitly according
to the variables described. Further analysis of this problem anatomy
is possible, including how it is displayed in regular research
articles and not just textbooks, as well as other areas of biology
(see, e.g., Brigandt and Love 2012).</p>

<h2><a id="ExpAppDev">3. Explanatory Approaches to Development</a></h2>

<p>
Explanations in developmental biology are usually causal, though
unlike standard mechanistic explanation there is a constant
acquisition of new causal capacities (in terms of constituent
entities, activities, and their organization) through development
(McManus 2012; Parkkinen 2014). Although much work remains in
characterizing different aspects of explanation in developmental
biology, there is no doubt that a difference making or manipulability
conception of causation (see the entry on
 <a href="../causation-mani/index.html">causation and manipulability</a>)
 provides a core element of the reasoning (Woodward 2003; Strevens
2009; Waters 2007a). Genetic explanations of development
 (<a href="#Gen">Section 3.1</a>),
 similar to what is seen in
 <a href="../molecular-genetics/index.html">molecular genetics</a>,
 work by identifying changes in the expression of genes and
interactions among their RNA and protein products that lead to changes
in the properties of morphological features during ontogeny (e.g.,
shape or size), while holding a variety of contextual variables fixed.
More recently, there has been growing interest in physical
explanations of development
 (<a href="#Phy">Section 3.2</a>)
 that involve appeals to mechanical forces due to geometrical
arrangements of mesoscale materials, such as fluid flow (Forgacs and
Newman 2005). Researchers agree on the phenomena that need to be
explained
 (<a href="#DevPhe">Section 1.2</a>
 and
 <a href="#EroOrg">Section 2.2</a>),
 but differ on whether physical rules or genetic factors are more or
less explanatory (Keller
 2002).<sup>[<a href="notes.html#note-8" id="ref-8">8</a>]</sup>
 The existence of two different types of causal explanations for
developmental phenomena poses an additional question about how they
might be combined into a more integrated explanatory framework
 (<a href="#IntAppGenPhy">Section 3.3</a>).</p>
 
<h3><a id="Gen">3.1 Genetics</a></h3>

<p>
Many philosophers have turned to explanations of development over the
past two decades in an effort to esteem or deflate claims about the
causal power of genes (Keller 2002; Neumann-Held and Rehmann-Sutter
2006; Rosenberg 2006; Robert 2004; Waters
 2007a).<sup>[<a href="notes.html#note-9" id="ref-9">9</a>]</sup>
 Genetic explanations touch the philosophical theme of reductionism
and appear to constitute the bulk of empirical success accruing to
developmental biology over the past several
 decades.<sup>[<a href="notes.html#note-10" id="ref-10">10</a>]</sup>
 Statements from developmental biologists reinforce this
perspective:</p>

<blockquote>

<p>
Developmental biology &hellip; deals with the process by which the
genes in the fertilized egg control cell behavior in the embryo and so
determine its pattern, its form, and much of its behavior &hellip;
differential gene activity controls development. (Wolpert et al. 1998:
v, 15)</p>
</blockquote>

<p>
These types of statements are sometimes amplified in appeals to a
genetic <em>program</em> for development.</p>

<blockquote>

<p>
[Elements of the genome] contain the sequence-specific code for
development; and they determine the particular outcome of
developmental processes, and thus the form of the animal produced by
every embryo. &hellip; Development is the execution of the genetic
program for construction of a given species of organism (Davidson
2006: 2,
 16).<sup>[<a href="notes.html#note-11" id="ref-11">11</a>]</sup></p>
 </blockquote>

<p>
At other times, statements concentrate on genetics as the primary
locus of causation in ontogeny: &ldquo;Developmental complexity is the
direct output of the spatially specific expression of particular gene
sets and it is at this level that we can address causality in
development&rdquo; (Davidson and Peter 2015: 2). Whether or not these
statements can be substantiated has been the subject of intense
 debate.<sup>[<a href="notes.html#note-12" id="ref-12">12</a>]</sup>
 The strongest claims about genetic programs or the genetic control of
development have empirical and conceptual drawbacks that include an
inattention to plasticity and the role of the environment, an
ambiguity about the locus of causal agency, and a reliance on
metaphors drawn from computer science (Gilbert and Epel 2009; Keller
2002; Moss 2002; Robert 2004). However, this leaves intact the
difference-making principle of genetic explanation exhibited in
molecular genetics (Waters 2007a), which yields more narrow and
precise causal claims under controlled experimental conditions, and is
applicable to diverse molecular entities that play causal roles during
development, such as regulatory RNAs, proteins, and environmental
signals. We can observe this briefly by reconsidering the example of
vertebrate cardiogenesis
 (<a href="#DevPhe">Section 1.2</a>).</p>
 
<p>
Are there problems with claiming that genes contain all of the
information (see the entry on
 <a href="../information-biological/index.html">biological information</a>)
 to form vertebrate hearts? Is there a genetic program in the DNA
controlling heart development? Are genes the primary supplier and
organizer of material resources for heart development, largely
determining the phenotypic outcome? Existing studies of heart
development have identified a role for fluid forces in specifying the
internal form of the heart (Hove et al. 2003) and its left/right
asymmetry (Nonaka et al. 2002). Biochemical gradients of extracellular
calcium are responsible for activating the asymmetric expression of
the regulatory gene <em>Nodal</em> (Raya et al. 2004) and inhibition
of voltage gradients scrambles normal asymmetry establishment (Levin
et al. 2002). Mechanical cues such as microenvironmental stiffness are
crucial for key transitions from migratory cells into organized sheets
during heart formation (Jackson et al. 2017). A number of genes are
clearly difference makers in these processes (Asp et al. 2019;
Srivastava 2006; Brand 2003; Olson 2006), but the conclusion that
genes carry all the information needed to generate form features of
the heart seems unwarranted. While it may be warranted empirically in
some cases to privilege DNA sequence differences as causal factors in
specific processes of ontogeny (Waters 2007a), such as hierarchically
organized networks of genetic difference makers explaining tissue
specification (Peter and Davidson 2011), the diversity of entities
appealed to in molecular genetics and the extent of their individual
and joint roles in specifying developmental outcomes implies that
debates about the meaning, scope, and power of genetic explanations
will continue (Griffiths and Stotz 2013). However, a shift away from
genetic programs and genetic determinism to DNA, RNA, and proteins as
difference makers that operate conjointly suggests that we
conceptualize other causal factors in a similar way.</p>

<h3><a id="Phy">3.2 Physics</a></h3>

<p>
Fluid flow, as a physical force, is also a difference maker during the
development of the heart, and ontogeny more generally, and
developmental biologists appeal to physical difference makers, which
are understood as factors in producing the morphological properties of
developmental phenomena (Forgacs and Newman 2005). A physical
causation approach was on display in the late 19th century work of
Wilhelm His (Hopwood 1999, 2000; Pearson 2018) and especially visible
in the early 20th century work of D&rsquo;Arcy Thompson and others
(Thompson 1992 [1942]; Keller 2002: ch. 2; Olby 1986). This occurred
in the milieu of increasing attention to the chromosomal theory of
inheritance and attempts to explore developmental phenomena via
classical genetic methods (Morgan 1923, 1926, 1934). Thompson appealed
to differential rates of growth and the constraints of geometrical
relationships to explain how organismal morphology originates. Visual
representations of abiotic, mechanical analogues provided the
plausibility, such as the shape of liquid splashes or hanging drops
for the cup and bell configurations of the free-swimming sexual stage
of jellyfish. If physical forces generated specific morphologies in
viscoelastic materials, then analogous morphologies in living species
should be explained in terms of physical forces operating on the
viscoelastic materials of the developing embryo. Yet morphogenetic
processes that produce the shape and structure of morphology have been
seen primarily, if not exclusively, in terms of genetics for the last
half-century. Physical approaches moved into the background as
molecular genetics approaches went from strength to strength (Fraser
and Harland 2000).</p>

<p>
The molecularization of experimental embryology is one of the most
striking success stories of contemporary biology as genes and genetic
interactions (e.g., in transcriptional networks and signaling
pathways; see
 <a href="#DevMech">Section 1.3</a>)
 were discovered to underlie specific details of differentiation,
morphogenesis, pattern formation, and growth when structure originates
during development. Genetic approaches predominate in contemporary
developmental biology and physical modes of causation are often
neglected. The frustration among researchers interested in physical
causation during embryogenesis has been palpable.</p>

<blockquote>

<p>
To the molecular types, a cause is a molecule or a gene. To explain a
phenomenon is to identify genes and characterize proteins without
which the phenomenon will fail or be abnormal. A molecule is an
explanation: a force is a description; to argue otherwise brings pity,
at best (Albert Harris to John Trinkaus, 12 March 1996; Source: Marine
Biological Laboratory Library Archives).</p>
</blockquote>

<p>
Despite this predominance of genetic explanatory approaches and the
frustration among researchers utilizing other approaches, a
groundswell of interest has been building around physical explanations
of development, especially in terms of their integration with genetic
explanations (Miller and Davidson 2013; Newman 2015). Some
philosophers have argued that the biomechanical modeling of physical
causal factors constitutes a rejection of certain forms of reductive
explanation in biology (Green and Batterman 2017).</p>

<h3><a id="IntAppGenPhy">3.3 Integrating Approaches: Genetics and Physics</a></h3>

<p>
Thompson held that physical forces were explanatory but inadequate in
isolation to account for the developmental origin of morphology;
heredity (genetics) was also a necessary causal
 factor.<sup>[<a href="notes.html#note-13" id="ref-13">13</a>]</sup>
 Yet Thompson was quick to highlight that mechanical modes of
causation might be neglected in the midst of growing attention to
heredity (genetics):</p>

<blockquote>

<p>
it is no less of an exaggeration if we tend to neglect these direct
physical and mechanical modes of causation altogether, and to see in
the characters of a bone merely the results of variation and of
heredity. (Thompson 1992 [1942]: 1023)</p>
</blockquote>

<p>
Despite this latter form of exaggeration manifesting itself through
much of the 20<sup>th</sup> century, an agenda to combine or integrate
the two approaches is now
 explicit.<sup>[<a href="notes.html#note-14" id="ref-14">14</a>]</sup></p>
 
<p>
There is no controversy about whether genetic and physical modes of
causation are at work simultaneously:</p>

<blockquote>

<p>
both the physics and biochemical signaling pathways of the embryo
contribute to the form of the organism. (Von Dassow et al. 2010:
1)</p>
</blockquote>

<p>
They are not competing causal explanations of the same phenomenon.
Explanations should capture how their productive interactions yield
developmental outcomes:</p>

<blockquote>

<p>
an increasing number of examples point to the existence of a
reciprocal interplay between expression of some developmental genes
and the mechanical forces that are associated with morphogenetic
movements. (Brouz&eacute;s and Farge 2004: 372)</p>
</blockquote>

<p>
Genetic causes can lead to physical causation and <em>vice versa</em>.
Physical causation brings about genetic causation through
mechanotransduction. Stretching, contraction, compression, fluid shear
stress, and other physical dynamics are sensed by different molecular
components inside and outside of cells that translate these
environmental changes into biochemical signals (Hoffman et al. 2011;
Wozniak and Chen 2009). Genetic causation brings about physical
causation by creating different physical properties of cells and
tissues through the presence, absence, or change in frequency of
particular proteins. For example, different patterns of expression for
cell adhesion molecules (e.g., cadherins) can lead to differential
adhesion across epithelial sheets of tissue and thereby generate phase
separations or compartments via surface tension variation (Newman and
Bhat 2008). If these modes of causation are not competing, then how
might one combine genetic and physical difference makers into an
integrated causal explanation? How much explanatory unity can be
achieved for this &ldquo;reciprocal interplay&rdquo;?</p>

<p>
Finding philosophical models for the explanatory integration of
genetics and physics remains an open question (Love 2017b).
Apportioning causal responsibility in the sense of determining
relative contributions (e.g., the composition of causal magnitudes
among different physical forces in Newtonian mechanics) is problematic
because this requires commensurability with respect to how causes
produce their effects (Sober 1988). In the context of causation
understood in terms of difference makers, the difficulty of
integration is a variation on a problem in causal reasoning identified
by John Stuart Mill and labeled the &ldquo;intermixture of
effects,&rdquo; which involves multiple causes contributing in a
blended fashion to yield an outcome.</p>

<blockquote>

<p>
This difficulty is most of all conspicuous in the case of
physiological phenomena; it being seldom possible to separate the
different agencies which collectively compose an organized body,
without destroying the very phenomena which it is our object to
investigate. (Mill 1843 [1974]: 456 [book 3, chapter 11, section 1,
paragraph 7])</p>
</blockquote>

<p>
Careful statistical methodology in experiments can answer whether one
type of difference maker accounts for more of the variation in the
effect variable for a particular population. But a ranking of causal
factors with respect to how much of a difference they made is not the
same as combining two modes of causation into an integrated account.
Another response is to dissolve the integration problem by reducing
all of the causal interactions to one of the two distinct modes,
thereby achieving a kind of explanatory unity (Rosenberg 2006).
However, this approach is eschewed by working biologists who take both
genetic and physical modes of causation as significant and not
reducible one to the other.</p>

<p>
A different strategy is integrative pluralism (Mitchell 2002). This
involves a two-step procedure for explaining complex phenomena whose
features are the result of multiple causes: (a) formulate idealized
models where particular causal factors operate in isolation
(&ldquo;theoretical modeling&rdquo;); and, (b) integrate idealized
models to explain how particular, concrete phenomena originate from
these causes in combination. This model is suggestive but also has key
drawbacks that include the fact that genetic causal reasoning in
developmental biology does not typically involve theoretical modeling
and the precise nature of the integration is underspecified.
Integration of genetic and physical difference makers in a single
mechanism offers a further possibility (Darden 2006; Craver 2007).
Although this valuably highlights the productive continuity between
difference makers through stages in a sequence (i.e., their reciprocal
interplay), it also has handicaps. These include:</p>

<ol>

<li>

<p>
Divergent approaches to measuring time. Instead of time &ldquo;in the
mechanism,&rdquo; time is measured with external standardized stages
(see below,
 <a href="#NorStaPhePla">Section 5.2</a>).
 Stages facilitate the study of different kinds of developmental
mechanisms, with different characteristic rates and durations for
their stages, within a common framework for a model organism (e.g.,
<em>Drosophila</em>), while also permitting conserved molecular
mechanisms to be studied in different species because the
corresponding mechanism description is not anchored to the temporal
sequence of the model organism.</p></li>

<li>

<p>
An expectation that mechanism descriptions &ldquo;bottom-out&rdquo; in
lowest level activities of molecular entities (Darden 2006). In the
case of combining genetic and physical difference makers, the
reciprocal interplay means that there is a studious avoidance of
bottoming out in one or the other mode of causation.</p></li>

<li>

<p>
The requirement of stable, compositional organization for mechanisms:
</p>

<blockquote>

<p>
Mechanistic explanations are constitutive or componential
explanations: they explain the behavior of the mechanism as a whole in
terms of the organized activities and interaction of its components.
(Craver 2007: 128) </p>
</blockquote>

<p>
But these mechanism descriptions are often embedded in different
developmental contexts (at different times in ontogeny) with distinct
compositional relations (within and between species). The reciprocal
interplay between genetic and physical difference makers is not
maintained precisely because these compositional differences alter
relationships of physical causation (fluid flow, tension, etc.; see
 <a href="#DevMech">Section 1.3</a>).
 Developmental biologists have been able to generalize relationships
of genetic causation (in terms of genetic mechanisms; see
 <a href="#DevMech">Section 1.3</a>)
 across species quite widely, but the attempt to combine these with
physical causation has necessitated narrowing the scope of the causal
claims.</p></li>
</ol>

<p>
Adequate philosophical models of the systematic dependence between
genetic and physical difference makers in ontogeny need to account for
how the temporal relations necessary for making causal claims are
anchored in an external periodization used by developmental
biologists. The imposition of different temporal scales can lead to
distinct factors being significant or salient, which matters for
ascertaining how different kinds of causes can be combined into
integrated explanations. One possibility is to juxtapose these
difference makers at distinct stages via experimental verification
such that they exhibit productive continuity within the constraints of
the external periodization (Love 2017b). This facilitates representing
symmetry between causal factors because genetic difference makers can
be placed before or after physical difference makers (and <em>vice
versa</em>). Although this does not provide a way to combine causal
magnitudes (as in vector addition from Newtonian mechanics), it offers
an explicit strategy for assigning responsibility among different
kinds of causes through the vehicle of temporal organization that goes
beyond ranking difference makers. The periodization serves as a
template from the practices of developmental biologists for providing
wholeness or unity to the different modes of causation to yield a kind
of integrated explanation of the morphology that results from a
sequence of developmental processes.</p>

<p>
Not all types of causal explanation involve an external periodization
and there are other ways to combine causes in order to produce more
integrated explanatory frameworks. One area where combined
explanations for developmental phenomena are being analyzed pertains
to mechanism descriptions and mathematical modeling in systems biology
(Brigandt 2013; Fagan 2013). For example, Fagan (2013: ch. 9) shows
how an integrated explanation emerges from a step-wise procedure that
starts with a detailed description of a molecular mechanism followed
by the formulation of an abstracted wiring diagram of component
interactions, which is then translated into a system of equations that
can account for changes in component interactions over time. Solutions
to these systems of equations and a mapping of solutions for the
interactions among components of the system onto the behavior of the
overall system within a shared landscape representation more
systematically explains cellular differentiation.</p>

<h2><a id="ModOrgForStuDev">4. Model Organisms for the Study of Development</a></h2>

<p>
Model organisms are central to contemporary biology and studies of
embryogenesis (Ankeny and Leonelli 2011; Steel 2008; Bier and McGinnis
2003; Davis 2004). Biologists utilize only a small number of species
to experimentally elucidate various properties of ontogeny (e.g.,
<em>C. elegans</em>, <em>Drosophila</em>, and <em>Brachydanio</em>
[zebrafish]; see
 <a href="#fig5">Figure 5</a>).
 These experimental models permit researchers to investigate
development in great depth and facilitate a precise dissection of
causal relationships. Critics have questioned whether these models are
good representatives of other species because of inherent biases
involved in their selection, such as rapid development and short
generation time (Bolker 1995), and problematic presumptions about the
conservation of gene functions and regulatory networks (Lynch 2009).
For example, <em>C. elegans</em> embryogenesis is not representative
of nematodes in terms of pattern formation and cell specification
(Schulze and Schierenberg 2011) and zebrafish appendage formation is a
poor proxy for the development of appendages in tetrapods (Metscher
and Ahlberg 1999).</p>

<div id="fig5" class="figure">
<img src="Figure5.jpg" style="width:200px;" alt="[a color photo showing from the side a 0.1 x 0.03 inch (2.5 x 0.8 mm) small male Drosophila melanogaster fly with red eyes facing left.]" />

<p>
<span class="figlabel">Figure 5:</span> <em>Drosophila
melanogaster</em> (the common fruit fly) is one of the standard model
organisms used in developmental biology.</p>
</div>

<p>
One response to this criticism is to emphasize the conserved genetic
mechanisms shared by all animals despite differences in developmental
phenomena (Gerhart and Kirschner 2007; Ankeny and Leonelli 2011; Weber
2005). Fruit flies may be unrepresentative in exhibiting syncytial
development, but they use the collinear expression of <em>Hox</em>
genes to specify their anterior-posterior body axis. This response
indicates that whether an entire model organism is representative
<em>per se</em> is too coarse-grained a criterion to capture the
rationale behind their use. We have to ask about representation
<em>with respect to what</em>, and some accounts have moved in this
direction. Jessica Bolker has distinguished exemplary and surrogate
modes of representation (Bolker 2009), where the former serve basic
research by exemplifying a larger group and the latter correspond to
models designed to provide indirect experimental access to otherwise
inaccessible phenomena, such as mouse models of human psychological
disorders (e.g., depression). Surrogate models are adopted in
biomedical contexts where the phenomena of interest are manifested in
humans. Most developmental biologists consider model organisms as
exemplars, not
 surrogates.<sup>[<a href="notes.html#note-15" id="ref-15">15</a>]</sup>
 Thus, in order to respond to a criticism of non-representativeness,
the criterion of representation must be explored in more
 detail.<sup>[<a href="notes.html#note-16" id="ref-16">16</a>]</sup></p>
 
<p>
A basic presumption about model organisms is that they bear
appropriate similarity relationships to larger groups of animals. This
presumption is an instantiation of what is discussed generally for
 <a href="../models-science/index.html">models in science</a>
 meant to represent phenomena. Model organisms represent developmental
phenomena in species that are either studied little or never studied
at all: &ldquo;we study flies and frogs as examples for the
development of animals in general&rdquo; (N&uuml;sslein-Volhard 2006:
87). One source of confidence in treating them as exemplars derives
from an inductive inference over discovered patterns of evolutionary
conservation with respect to developmental phenomena (e.g.,
gastrulation or somite formation). If all or most model organisms
share a developmental feature, then all or most animals will share the
feature. This inference can be circumscribed more or less narrowly
(e.g., if all or most vertebrate model organisms share somite
formation, then all or most vertebrates will share it).</p>

<p>
As a consequence of this confidence, the model organism
(&ldquo;source&rdquo;) can represent these other unstudied species
(&ldquo;targets&rdquo;). This basic distinction between the model or
source and the phenomena or target it is supposed to represent is
ubiquitous in reasoning with model organisms (Ankeny and Leonelli
2011). Zebrafish is a model or representation of vertebrate
development, the target phenomena, because we expect to learn about
vertebrate development generally by studying ontogeny in zebrafish
specifically. We do not invest time and resources into zebrafish as a
model organism only because we are interested in zebrafish.
Researchers plan to make claims about somite formation from
observations in zebrafish that will apply to somite formation in other
vertebrates that we will never have the time or money to
investigate.</p>

<p>
Developmental biologists often speak of investigating mechanisms that
account for phenomena in ontogeny (see
 <a href="#DevMech">Section 1.3</a>),
 and focus on conserved genetic and cellular mechanisms in model
organisms (Gerhart and Kirschner 2007; Ankeny and Leonelli 2011; Weber
2005). This suggests a distinction between representation with respect
to developmental phenomena and representation with respect to genetic
and cellular mechanisms operating in development. If we are interested
in explaining how hearts (<em>phenomena</em>) develop, then we might
investigate the molecular or cellular <em>mechanisms</em> occurring in
the heart field during zebrafish ontogeny. Some of these mechanisms
could be conserved even though the phenomena are not.
<em>Drosophila</em> has only one cardiac cell type, no neural crest
cells, and a heart with no atrial or ventricular chamber morphology
(Kirby 1999). However, cardiogenesis in all invertebrates and
vertebrates investigated thus far depends essentially on the
expression of the homeobox gene <em>Nkx2-5</em>/<em>tinman</em>
(Gajewski and Schulz 2002). The reverse situation also can hold:
similar phenomena may be manifested but genetic and cellular
mechanisms might differ. Amphibians form a neural tube (neurulation)
through a process of invagination (the folding of an epithelial
sheet), whereas teleost fishes form a neural tube via cavitation (the
hollowing out of a block of tissue via cell death). The neural tube is
homologous across vertebrates (i.e., a conserved phenomenon), but the
cellular and genetic mechanisms involved in invagination versus
cavitation are distinct (Davies 2013: ch. 4).</p>

<p>
The distinction between phenomena and mechanisms assumes specificity;
i.e., there are specific phenomena (somite formation in vertebrates)
or mechanisms (collinear <em>Hox</em> gene expression) in view when
judging the relationship between source (model) and target. But animal
development consists of a multitude of different processes that
involve a host of different mechanisms. Therefore, another distinction
operating in the representational criterion pertains to questions of
<em>specificity</em> versus <em>variety</em> when selecting and using
model organisms. A model might represent one type of target phenomena
(differentiation or growth) or mechanism (cell signaling or cell
cycling) but not others&mdash;specificity&mdash;or may do so better or
worse with respect to particular types of phenomena or mechanisms. A
model might represent several types of target phenomena and mechanisms
simultaneously&mdash;variety&mdash;with variability in how each type
is represented. Trade-offs exist with respect to how well different
phenomena or mechanisms are co-instantiated in a model organism. Note
that experimental organisms may be selected with respect to variety
and specificity simultaneously, such as when a biologist working on a
specific phenomenon intends to work on others using the same model in
the future. They also may be selected with one or the other of these
two aspects predominant. A model might be desirable if it has
representational variety in both mechanisms and phenomena even if it
is not the best representative for every specific mechanism or
phenomenon. Conversely, a model organism might be desirable if it is
the best representative for a specific mechanism despite being a very
poor model for other phenomena or mechanisms. Variety is indicative of
the &ldquo;whole organism&rdquo; being the
 model.<sup>[<a href="notes.html#note-17" id="ref-17">17</a>]</sup>
 A further distinction can be introduced between &ldquo;model
organisms&rdquo; and &ldquo;experimental organisms&rdquo; (Ankeny and
Leonelli 2011) or &ldquo;general model organisms&rdquo; and
&ldquo;Krogh-principle model organisms&rdquo; (Love 2010). General
model organisms are selected and used with the variety aspect of the
representational criterion preeminent; experimental or Krogh-principle
model organisms are selected and used with specificity preeminent.</p>

<p>
Other issues relevant to the representation criterion include how
individual cells or cell types serve as developmental models (Fagan
2016), how developmental mechanisms in different model organisms are
compared and evaluated (Yoshida forthcoming), how the use of model
organisms constitutes an example of case-based reasoning (Ankeny
2012), and how model organisms involve idealizations or known
departures from features present in the model&rsquo;s target as the
result of laboratory cultivation (Ankeny 2009;
 <a href="#NorStaPhePla">Section 5.2</a>).
 Additionally, the question of representation is not the only one
germane to understanding model organism use. Because model organisms
are utilized for experimental intervention, questions of
representation must be juxtaposed with questions of manipulation (see
the supplement on
 <a href="supplement.html">Model Organisms and Manipulation</a>).</p>
 
<h2><a id="DevEvo">5. Development and Evolution</a></h2>

<p>
The relationships that obtain between development and evolution are
complicated and under ongoing investigation (for a review, see Love
2015). Two main axes dominate within a loose conglomeration of
research programs (Raff 2000; M&uuml;ller 2007): (a) the evolution of
development, or inquiry into the pattern and processes of how ontogeny
varies and changes over time; and, (b) the developmental basis of
evolution, or inquiry into the causal impact of ontogenetic processes
on evolutionary trajectories&mdash;both in terms of constraint and
facilitation. Two examples where the concepts and practices of
developmental and evolutionary biology intersect are treated here: the
problematic appeal to functional homology in developmental genetics
that is meant to underwrite evolutionary generalizations about
ontogeny
 (<a href="#FunHomDevGen">Section 5.1</a>)
 and the tension between using normal stages for developmental
investigation and determining the evolutionary significance of
phenotypic plasticity
 (<a href="#NorStaPhePla">Section 5.2</a>).
 These cases expose some of the philosophical issues inherent in how
development and evolution can be related to one another.</p>

<h3><a id="FunHomDevGen">5.1 Functional Homology in Developmental Genetics</a></h3>

<p>
The conserved role of <em>Hox</em> genes in axial patterning is
referred to as functionally homologous across animals (Manak and Scott
1994), over and above the relation of structural homology that obtains
between DNA sequences. And yet &ldquo;functional homology&rdquo; is a
contradiction in terms (Abouheif et al. 1997) because the definition
of a homologue is &ldquo;the same organ in different animals under
every variety of form and function&rdquo; (Owen 1843: 379)&mdash;the
descendant, evolutionary distinction between homology (structure) and
analogy (function) is founded on this recognition. Therefore, the idea
of functional homology appears theoretically confused and there is a
conceptual tension in its use by molecular developmental
biologists.</p>

<div id="fig6" class="figure">
<img src="Figure6.jpg" alt="[three skeletons each with the left wing outstretched and the outline of the wing shaded in. The skeleton labeled 1 is of a pterodactyl; that labeled 2 is a bat; and that of 3 is a bird.]" />

<p>
<span class="figlabel">Figure 6:</span> Vertebrate wings are
homologous as forelimbs; they are derived by common descent from the
same structure. The function of vertebrate wings (i.e., flight) is
analogous; although the wings fulfill similar functions, their role in
flight has evolved separately.</p>
</div>

<p>
The reference to &ldquo;organ&rdquo; in Owen&rsquo;s definition is
indicative of a structure (an entity) found in an organism that may
vary in its shape and composition (form) or what it is for (function)
in the species where it occurs. Translated into an evolutionary
context, sameness is cashed out by reference to common ancestry. Since
structures also can be similar by virtue of natural selection
operating in similar environments, homology is contrasted with
analogy. Homologous structures are the same by virtue of descent from
a common ancestor, regardless of what functions these structures are
involved in, whereas analogous structures are similar by virtue of
selection processes favoring comparable functional outcomes,
regardless of common descent
 (<a href="#fig6">Figure 6</a>).
 </p>

<p>
This is what makes similarity of function an especially problematic
criterion of homology (Abouheif et al. 1997). Because functional
similarity is the appropriate relation for analogy, it is not
necessary for analogues to have the same function as a consequence of
common ancestry&mdash;similarity despite different origins suffices
(Ghiselin 2005). Classic cases of analogy involve taxa that do not
share a recent common ancestor that exhibits the structure, such as
the external body morphology of dolphins and tuna (Pabst 2000). Thus,
functional homology seems to be a category error because what a
structure does should not enter into an evaluation of homologue
correspondence and similarity of function is often the result of
adaptation via natural selection to common environmental demands, not
common ancestry.</p>

<p>
Although we might be inclined to simply prohibit the terminology of
functional homology, its widespread use in molecular and developmental
biology should at least make us
 pause.<sup>[<a href="notes.html#note-18" id="ref-18">18</a>]</sup>
 While it is important to recognize this pervasive practice, some
occurrences may be illicit. Swapping structurally homologous genes
between species to rescue mutant or null phenotypes is not a genuine
criterion of functional homology, especially when there is little or
no attention to establishing a phylogenetic context. This makes a
number of claims of functional homology suspect. To not run afoul of
the conceptual tension, explicit attention must be given to the
meaning of &ldquo;function.&rdquo; Biological practice harbors at
least four separate meanings of function (Wouters 2003, 2005):
activity (what something does), causal role (contribution to a
capacity), fitness advantage or viability (value of having something),
and selected effect or etiology (origination and maintenance via
natural selection). Debate has raged about which of them (if any) is
most appropriate for different aspects of biological and psychological
reasoning or most general in scope (i.e., what makes them all function
concepts?) (see discussion in Garson 2016). Here the issue is whether
we can identify a legitimate concept of homology of function.</p>

<p>
If we are to avoid mixing homology and analogy, then the appropriate
notion of function cannot be based on selection history, which is
allied with the concept of analogy and concerns a particular variety
of function. Similarly, viability interpretations concentrate on
features where the variety of function is critical because of
conferred survival advantages. Any interpretation of function that
relies on a particular variety of function (because it was selected or
because it confers viability) clashes with the demand that homology
concern something &ldquo;under every variety of form and
function.&rdquo; A causal role interpretation emphasizes a systemic
capacity to which a function makes a contribution. It too focuses on a
particular variety of function, though in a way different from either
selected effect or viability interpretations. Only an activity
interpretation (&lsquo;what something does&rsquo;) accents the
function itself, apart from its specific contribution to a systemic
capacity and position in a larger context. Therefore, the most
appropriate meaning to incorporate into homology of function is
&ldquo;<em>activity</em>-function&rdquo; because it is at least
possible for activity-functions to remain constant under every
variety. An evaluation of sameness due to common ancestry is made
separately from the role the function plays (or its use), whether
understood in terms of a causal role, a fitness advantage, or a
history of
 selection.<sup>[<a href="notes.html#note-19" id="ref-19">19</a>]</sup>
 <em>Activity</em>-functions can be put to different uses while being
shared via common descent (i.e., homologous). More precisely, homology
of function can be defined as the same activity-function in different
animals under every variety of form and use-function (Love 2007). This
unambiguously removes the tension that plagued functional
homology.</p>

<p>
Careful discussions of regulatory gene function in development and
evolution recognize something akin to the distinction between
activity- and use-function (i.e., between what a gene does and what it
is for in some process within the organism).</p>

<blockquote>

<p>
When studying the molecular evolution of regulatory genes, their
biochemical and developmental function must be considered separately.
The biochemical function of <em>PAX-6</em> and <em>eyeless</em> are as
general transcription factors (which bind and activate downstream
genes), but their developmental function is their specific involvement
in eye morphogenesis (Abouheif 1997: 407).</p>
</blockquote>

<p>
The biochemical function is the activity-function and the
developmental function is the use-function. This distinction helps to
discriminate between divergent evolutionary trajectories. Biochemical
(activity-functions) of genes are often conserved (i.e., homologous),
while simultaneously being available for co-option to make causal role
contributions (use-functions) to distinct developmental processes. The
same regulatory genes are evolutionarily stable in terms of
activity-function and evolutionarily labile in terms of
 use-function.<sup>[<a href="notes.html#note-20" id="ref-20">20</a>]</sup>
 By implication, claims about use-function homology for genes
<em>qua</em> developmental function are suspect compared to those
concerning activity-function homology for genes <em>qua</em>
biochemical function because developmental functions are more likely
to have changed as phylogenetic distance increases.</p>

<p>
The distinction between biochemical (activity) function and
developmental (use) function is reinforced by the hierarchical aspects
of homology (Hall 1994). A capacity defining the use-function of a
regulatory gene at one level of organization, such as axial
patterning, must be considered as an activity-function itself at
another level of organization, such as the differentiation of serially
repeated elements along a body axis. (Note that
 &ldquo;<a href="../levels-org-biology/index.html">level of organization</a>&rdquo;
 need not be compositional and thus the language of
&ldquo;higher&rdquo; and &ldquo;lower&rdquo; levels may be
inappropriate.) The developmental roles of <em>Hox</em> genes in axial
patterning may be conserved by virtue of their biochemical
activity-function homologies but <em>Hox</em> genes are not
use-function homologues because of these developmental roles. Instead
of focusing on the activity of a gene component and its causal role in
axial patterning, we shift to the activity of axial patterning and its
causal role elsewhere (or elsewhen) in embryonic development.</p>

<p>
Introducing a conceptually legitimate idea of homology of
activity-function is not about keeping the ideas of developmental
biology tidy. It assists in the interpretation of evidence and
circumscribes the inferences drawn. For example, <em>NK-2</em> genes
are involved in mesoderm specification, which underlies muscle
morphogenesis. In <em>Drosophila</em>, the expression of a particular
<em>NK-2</em> gene (<em>tinman</em>) is critical for both cardiac and
visceral mesoderm development. If <em>tinman</em> is knocked out and
transgenically replaced with its vertebrate orthologue,
<em>Nkx2-5</em>, only visceral mesoderm specification is rescued; the
regulation of cardiac mesoderm is not (Ranganayakulu et al. 1998). A
region of the vertebrate protein near the 5&prime; end of the
polypeptide differs enough to prevent appropriate regulation in
cardiac morphogenesis. The homeodomains (stretches of sequence that
confer DNA binding) for vertebrate <em>Nkx2-5</em> and
<em>Drosophila</em> <em>tinman</em> are interchangeable. The inability
of <em>Nkx2-5</em> to rescue cardiac mesoderm specification is not
related to the activity-function of differential DNA binding. One
component of the orthologous (homologous) proteins in both species
retains an activity-function homology related to visceral mesoderm
specification but another component (not the homeodomain) has
diverged. This homeobox gene does not have a single use-function (as
expected), but it also does not have a single activity-function. Any
adequate evaluation of these cases must recognize a more fine-grained
decomposition of genes into working units to capture genuine
activity-function conservation. We can link activity-function
homologues directly to structural motifs within a gene, but there is
not necessarily a single activity-function for an entire open reading
frame.</p>

<p>
Defusing the conceptual tensions between developmental and
evolutionary biology with respect to homology of function has a direct
impact on the causal generalizations and inferences made from model
organisms
 (<a href="#ModOrgForStuDev">Section 4</a>).
 Activity-function homology directs our attention to the stability or
conservation of activities. This conservation is indicative of when
the study of mechanisms in model organisms will produce robust and
stable generalizations
 (<a href="#DevMech">Section 1.3</a>).
 The widespread use of functional homology in developmental biology is
aimed at exactly this kind of question, which explains its persistence
in experimental biology despite conceptual ambiguities.
Generalizations concerning molecular signaling cascades are
underwritten by the coordinated biochemical activities in view, not
the developmental roles (though sometimes they may coincide). Thus,
activity-function details about a signaling cascade gleaned from a
model organism can be generalized via homology to other unstudied
organisms even if the developmental role varies for the
activity-function in other species.</p>

<h3><a id="NorStaPhePla">5.2 Normal Stages and Phenotypic Plasticity</a></h3>

<p>
All reasoning strategies combine distinctive strengths alongside of
latent weaknesses. For example, decomposing a system into its
constituents to understand the features manifested by the system
promotes a dissection of the causal interactions of the localized
constituents, while downplaying interactions with elements external to
the system (Wimsatt 1980; Bechtel and Richardson 1993). Sometimes the
descriptive and explanatory practices of the sciences are successful
precisely because they intentionally ignore aspects of natural
phenomena or use a variety of approximation techniques. Idealization
is one type of reasoning strategy that scientists use to describe,
model, and explain that purposefully departs from features known to be
present in nature. For example, the interior space of a cell is often
depicted as relatively empty even though intracellular space is known
to be crowded (Ellis 2001); the variable of cellular volume takes on a
value that is known to be false (i.e., relatively empty).
Idealizations involve knowingly ignoring variations in properties or
excluding particular values for variables, in a variety of different
ways, for descriptive and explanatory purposes (Jones 2005; Weisberg
2007).</p>

<p>
&ldquo;Normal development&rdquo; is conceptualized through strategies
of abstraction that manage variation inherent within and across
developing organisms (Lowe 2015, 2016). The study of ontogeny in model
organisms
 (<a href="#ModOrgForStuDev">Section 4</a>)
 is usually executed by establishing a set of normal stages for
embryonic development (see Other Internet Resources). A developmental
trajectory from fertilized zygote to fully-formed adult is broken down
into distinct temporal periods by reference to the occurrence of major
events, such as fertilization, gastrulation, or metamorphosis (Minelli
2003: ch. 4; see
 <a href="#DevPhe">Section 1.2</a>).
 This enables researchers in different laboratory contexts to have
standardized comparisons of experimental results (Hopwood 2005, 2007).
They are critical to large communities of developmental biologists
working on well-established models, such as chick (Hamburger and
Hamilton 1951) or zebrafish (Kimmel et al. 1995): &ldquo;Embryological
research is now unimaginable without such standard series&rdquo;
(Hopwood 2005: 239). These normal stages are a form of idealization
because they intentionally ignore kinds of variation in development,
including variation associated with environmental variables. While
facilitating the study of particular causal relationships, this means
that specific kinds of variation in developmental features that might
be relevant to evolution are minimized in the process of rendering
ontogeny experimentally tractable (Love 2010).</p>

<p>
Phenotypic plasticity is a ubiquitous biological phenomenon. It
involves the capacity of a particular genotype to generate phenotypic
variation, often in the guise of qualitatively distinct phenotypes, in
response to differential environmental cues (Pigliucci 2001; DeWitt
and Scheiner 2004; Kaplan 2008; Gilbert and Epel 2009). One familiar
example is seasonal caterpillar morphs that depend on different
nutritional sources (Greene 1989). Some of the relevant environmental
variables include temperature, nutrition, pressure/gravity, light,
predators or stressful conditions, and population density (Gilbert and
Epel 2009). The <em>reaction norm</em> is a summary of the range of
phenotypes, whether quantitatively or qualitatively varying, exhibited
by organisms of a given genotype for different environmental
conditions. When the reaction norm exhibits discontinuous variation or
bivalent phenotypes (rather than quantitative, continuous variation),
it is often labeled a <em>polyphenism</em>
 (<a href="#fig7">Figure 7</a>).</p>
 
<div id="fig7" class="figure avoid-break">
<img src="Figure7.png" style="width:300px;" alt="[two color photos of leafed twigs each with a well camouflaged caterpillars (Biston betularia) looking like a branch, one green (on willow, right) and one brown (on birch, left).]" />

<p>
<span class="figlabel">Figure 7:</span> A color polyphenism in
American Peppered Moth caterpillars that represents an example of
phenotypic plasticity.</p>
</div>

<p>
Phenotypic plasticity has been of recurring interest to biological
researchers and controversial in evolutionary theory. Extensive study
of phenotypic plasticity has occurred in the context of quantitative
genetic methods and phenotypic selection analyses, where the extent of
plasticity in natural populations has been demonstrated and
operational measures delineated for its detection (Scheiner 1993;
Pigliucci 2001). Other aspects of plasticity require different
investigative methods to ascertain the sources of plasticity during
ontogeny, the molecular genetic mechanisms that encourage plasticity,
and the kinds of mapping functions that exist between the genotype and
phenotype (Pigliucci 2001; Kirschner and Gerhart 2005: ch. 5). These
latter aspects, the origin of phenotypic variation during and after
ontogeny, are in view at the intersection of development and
evolution: How do molecular genetic mechanisms produce (or reduce)
plasticity? What genotype-phenotype mapping functions are prevalent or
rare? Does plasticity contribute to the origination of evolutionary
novelties (Moczek et al. 2011; West-Eberhard 2003)?</p>

<p>
In order to evaluate these questions experimentally, researchers need
to alter development through the manipulation of environmental
variables and observe how a novel phenotype can be established within
the existing plasticity of an organism (Kirschner and Gerhart 2005:
ch. 5). This manipulation could allow for the identification of
patterns of variation through the reliable replication of particular
experimental alterations within different environmental regimes.
However, without measuring variation across different environmental
regimes, you cannot observe phenotypic plasticity. These measurements
are required to document the degree of plasticity and its patterns for
a particular trait, such as qualitatively distinct morphs. An
evaluation of the significance of phenotypic plasticity for evolution
requires answers to questions about where plasticity emerges, how
molecular genetic mechanisms are involved in the plasticity, and what
genotype-phenotype relations obtain.</p>

<p>
Developmental stages intentionally ignore variation associated with
phenotypic plasticity. Animals and plants are raised under stable
environmental conditions so that stages can be reproduced in different
laboratory settings and variation is often viewed as noise that must
be reduced or eliminated if one is to understand how development works
(Frankino and Raff 2004). This practice also encourages the selection
of model organisms that exhibit less plasticity (Bolker 1995). The
laboratory domestication of a model organism may also reduce the
amount or type of observable phenotypic variation (Gu et al. 2005),
though laboratory domestication also can increase variation (e.g., via
inbreeding). Despite attempts to reduce variation by controlling
environmental factors, some of it always remains (Lowe 2015) and is
displayed by the fact that absolute chronology is not a reliable
measure of time in ontogeny, and neither is the initiation or
completion of its different parts (Mabee et al. 2000; Sheil and
Greenbaum 2005). Developmental stages allow this recalcitrant
variation to be effectively ignored by judgments of embryonic
typicality. Normal stages also involve assumptions about the causal
connections between different processes across sequences of stages
(Minelli 2003: ch. 4). Once these stages have been constructed, it is
possible to use them as a visual standard against which to recognize
and describe variation as a deviation from the norm (DiTeresi 2010;
Lowe 2016). But, more typically, variation ignored in the construction
of these stages is also ignored in the routine consultation of the
stages in day-to-day research contexts (Frankino and Raff 2004).</p>

<p>
Normal stages fulfill a number of goals related to descriptive and
explanatory endeavors that developmental biologists engage in (Kimmel
et al. 1995). They yield a way to measure experimental replication,
enable consistent and unambiguous communication among researchers,
especially if stages are founded on commonly observable morphological
features, facilitate accurate predictions of developmental phenomena,
and aid in making comparisons or generalizations across species. As
idealizations of ontogeny, normal stages allow for a classification of
developmental events that is comprehensive with suitably sized and
relatively homogeneous stages, reasonably sharp boundaries between
stages, and stability under different investigative conditions
(Dupr&eacute; 2001), which encourages more precise explanations within
particular disciplinary approaches (Griesemer 1996). Idealizations
also can facilitate abstraction and generalization, both of which are
a part of extrapolating findings from the investigative context of a
model organism to other domains (Steel 2008; see
 <a href="#ModOrgForStuDev">Section 4</a>
 and
 <a href="#FunHomDevGen">5.1</a>).</p>
 
<p>
There are various weaknesses associated with normal stages that
accompany the fulfillment of these investigative and explanatory
goals. Key morphological indicators sometimes overlap stages,
terminology that is useful for one purpose may be misleading for
another, particular terms can be misleading in cross-species
comparisons, and manipulation of the embryo for continued observation
can have a causal impact on ontogeny. Avoiding variability in stage
indicators can encourage overlooking the significance of this
variation, or at least provide a reason to favor its minimization.</p>

<p>
Thus, there are good reasons for adopting normal stages to periodize
model organism ontogeny, and these reasons help to explain why their
continued use yields empirical success. However, similar to other
standard (successful) practices in science, normal stages are often
taken for granted, which means their biasing effects are neglected
(Wimsatt 1980), some of which are relevant to evolutionary questions
(e.g., systematically underestimating the extent of variation in a
population). This is critical to recognize because the success of a
periodization is not a function of the eventual ability to relax the
idealizations; periodizations are not slowly corrected so that they
become less idealized. Instead, new periodizations are constructed and
used alongside the existing ones because different idealizations
involve different judgments of typicality that serve diverse
descriptive and explanatory aims. In addition to the systematic biases
involved in developmental staging, most model organisms are poorly
suited to inform us about how environmental effects modulate or
combine with genetic or other factors in development&mdash;they make
it difficult to discover details about mechanisms underlying reaction
norms. Short generation times and rapid development are tightly
correlated with insensitivity to environmental conditions through
various mechanisms such as prepatterning (Bolker 1995).</p>

<p>
The tension between the specific practice of developmental staging in
model organisms and uncovering the relevance of variation due to
phenotypic plasticity for evolution can be reconstructed as an
argument.</p>

<ol>

<li>Variation due to phenotypic plasticity is a normal feature of
ontogeny.</li>

<li>The developmental staging of model organisms intentionally
downplays variation in ontogeny associated with the effects of
environmental variables (e.g., phenotypic plasticity) by strictly
limiting the range of values for environmental variables and by
removing variation in characters utilized to establish the
comprehensive periodization.</li>

<li>Therefore, using model organisms with specified developmental
stages will make it difficult, if not impossible, to observe patterns
of variation due to phenotypic plasticity.</li>
</ol>

<p>
Although this tension obtains even if the focus is not on evolutionary
questions, sometimes encouraging developmental biologists to interpret
absence of evidence as evidence of the developmental insignificance of
phenotypic plasticity, it is exacerbated for evolutionary researchers.
The documentation of patterns of variation is precisely what is
required to gauge the evolutionary significance of phenotypic
plasticity. Practices of developmental staging in model organisms can
retard our ability to make either a positive or negative assessment.
Developmental staging, in conjunction with the properties of model
organisms, tends to encourage a negative assessment of the
evolutionary importance of phenotypic plasticity because the variation
is not manifested and documented, and therefore is unlikely to be
reckoned as substantive. Idealizations involving normal stages
discourage a robust experimental probing of phenotypic plasticity,
which is an obstacle to determining its evolutionary significance.</p>

<p>
The consequences of this tension for the intersection of development
and evolution are two-fold. First, the most powerful experimental
systems for studying development are set up to minimize variation that
may be critical to comprehending how evolutionary processes occur in
nature. Second, if evolutionary investigations revolve around a
character that was assessed for typicality to underwrite the temporal
partitions that we call stages, then much of the variation in this
character was conceptually removed as a part of rendering the model
organism experimentally
 tractable.<sup>[<a href="notes.html#note-21" id="ref-21">21</a>]</sup></p>
 
<p>
The identification of drawbacks that accompany strategies of
idealization used to study development invites consideration of ways
to address the liabilities identified (Love 2006). We can construct a
principled perspective on how to address these liabilities by adding
three further premises:</p>

<ol start="4">

<li>Reasoning strategies involving idealization, such as (2), are
necessary to the successful prosecution of biological investigations
of ontogeny.</li>

<li>Therefore, compensatory tactics should be chosen in such a way as
to specifically redress the blind spots arising from the kind of
idealizations utilized.</li>

<li>Given (1)&ndash;(3), compensatory tactics must be related to the
effects of ignoring variation due to phenotypic plasticity that result
from the developmental staging of model organisms.</li>
</ol>

<p>
At least two compensatory tactics can promote observations of
variation due to phenotypic plasticity that is ignored when
developmental stages are constructed for model organisms: the
employment of diverse model organisms and the adoption of alternate
periodizations.</p>

<p>
Variation often will be observable in non-standard model organisms
because experimental organisms that do not have large communities
built around them are less likely to have had their embryonic
development formally staged, and thus the effects of idealization on
phenotypic plasticity are not operative. In turn, researchers are
sensitized to the ways in which these kinds of variation are being
muted in the study of standard models. Stages can be used then as
visual standards to identify variation as deviations from a norm and
thereby characterize patterns of
 variability.<sup>[<a href="notes.html#note-22" id="ref-22">22</a>]</sup></p>
 
<p>
A second compensatory tactic is the adoption of alternative
periodizations. This involves choosing different characters to
construct new temporal partitions, thereby facilitating the
observation of variation with respect to characteristics previously
stabilized in the normal stage periodization. These alternative
periodizations often divide a subset of developmental events according
to processes or landmarks that differ from those used to construct the
normal stages, and they may not map one-one onto the existing normal
stages, especially if they encompass events beyond the trajectory from
fertilization to a sexually mature adult. This lack of isomorphism
between periodizations also will be manifested if different measures
of time are utilized, whether sequence (event ordering) or duration
(succession of defined intervals), and whether sequences or durations
are measured relative to one another or against an external standard,
such as absolute chronology (Reiss 2003; Colbert and Rowe 2008). These
incompatibilities prevent assimilating the alternative periodizations
into a single, overarching staging scheme. In all of these cases,
idealization is involved and therefore each new periodization is
subject to the liabilities of ignoring kinds of variation. However,
alternative periodizations require choosing different characters to
stabilize and typify when defining its temporal partitions, which
means different kinds of variation will be exposed than were
previously
 observable.<sup>[<a href="notes.html#note-23" id="ref-23">23</a>]</sup></p>
 
<p>
The compensatory tactics of employing a diversity of model organisms
and adopting alternative periodizations may be conceptually
appropriate for addressing how the practice of developmental staging
has an impact on the detection of phenotypic plasticity, but this does
not remove associated costs (human, financial, and otherwise) or
controversy. The advantages of a single, comprehensive periodization
for a general model organism (e.g., zebrafish normal stages) must be
weighed in light of the advantages of alternative, process-specific
periodizations. However, by openly scrutinizing these practices in
relation to the phenomenon of interest and recognizing both advantages
and drawbacks involved in the idealizations utilized, developmental
and evolutionary biologists are better positioned to offer systematic
descriptions and comprehensive explanations of biological
phenomena.</p>

<h2><a id="Con">6. Conclusion</a></h2>

<p>
This entry has only sampled a small portion of work relevant to the
import and promise of conceptual reflection on the epistemology of
developmental biology. Much more could be said about each of the above
domains, such as a more fine-grained analysis of how normal stages
operate as types in developmental biology (DiTeresi 2010; Lowe 2016).
Additionally, little has been said about how evidence works in
developmental biological experimentation or differences between
confirmatory and exploratory experimentation (Hall 2005;
O&rsquo;Malley 2007; Waters 2007b), nor have I treated the role of
metaphors and models that characterize key practices in developmental
biology (Fagan 2013; Keller 2002). The latter have been perspicuously
analyzed via increased attention to the details of particular research
programs. Finally, nothing has been said about the metaphysical
implications of developmental phenomena (a key input for
 <a href="../aristotle-metaphysics/index.html">Aristotle&rsquo;s metaphysics</a>).
 Concepts of potentiality are very natural in descriptions of
embryological phenomena (e.g., the pluripotency of stem cells or the
potential of a germ layer to yield different kinds of tissue lineages)
and some have argued that empirical advances in developmental biology
support a new form of essentialism about biological natural kinds
(Austin 2019). This bears on how we understand dispositions (see the
entry on
 <a href="../dispositions/index.html">dispositions</a>)
 because the triggering conditions are often complex and multiply
realized (including manifestations without a trigger), as well as the
fact that cells exhibit dispositions with multiple possible
manifestations (cell types) in specific sequential orderings
(H&uuml;ttemann and Kaiser 2018; Laplane 2016). Metaphysical issues
also arise in the context of human developmental biology, such as how
to understand the ontology of pregnancy (Kingma 2018; Sidzinska 2017).
Thus, developmental biology displays not only a rich array of material
and conceptual practices that can be analyzed to better understand the
scientific reasoning exhibited in experimental life science, but also
points in the direction of new ideas for metaphysics, especially when
that endeavor explicitly considers the input of empirically successful
sciences.</p>
</div>

<div id="bibliography">

<h2><a id="Bib">Bibliography</a></h2>

<h3><a id="Ref">References</a></h3>

<ul class="hanging">

<li>Abouheif, E., 1997, &ldquo;Developmental genetics and homology: a
hierarchical approach&rdquo;, <em>Trends in Ecology and
Evolution</em>, 12: 405&ndash;408.</li>

<li>Abouheif, E., M. Akam, W.J. Dickinson, P.W.H. Holland, A. Meyer,
N.H. Patel, R.A. Raff, V.L. Roth, and G.A. Wray, 1997, &ldquo;Homology
and developmental genes&rdquo;, <em>Trends in Genetics</em>, 13:
432&ndash;433.</li>

<li>Ankeny, R.A., 2009, &ldquo;Model organisms as fictions&rdquo;, in
<em>Fictions in Science: Philosophical Essays on Modeling and
Idealization</em>, M. Su&aacute;rez (ed.), 193&ndash;204. New York and
London: Routledge, Tayor &amp; Francis Group.</li>

<li>&ndash;&ndash;&ndash;, 2012, &ldquo;Detecting themes and
variations: the use of cases in developmental biology&rdquo;,
<em>Philosophy of Science</em>, 79: 644&ndash;654.</li>

<li>Ankeny, R.A., and S. Leonelli, 2011, &ldquo;What&rsquo;s so
special about model organisms?&rdquo;, <em>Studies in History and
Philosophy of Science</em>, 42: 313&ndash;323.</li>

<li>Asp, M., S. Giacomello, L. Larsson, C. Wu, D. F&uuml;rth, X. Qian,
E. W&auml;rdell, J. Custodio, J. Reimeg&aring;rd, F. Salm&eacute;n, C.
&Ouml;sterholm, P. L. St&aring;hl, E. Sundstr&ouml;m, E.
&Aring;kesson, O. Bergmann, M. Bienko, A. M&aring;nsson-Broberg, M.
Nilsson, C. Sylv&eacute;n and J. Lundeberg, 2019, &ldquo;A
spatiotemporal organ-wide gene expression and cell atlas of the
developing human heart&rdquo;, <em>Cell</em>, 179:
1647&ndash;1660.</li>

<li>Austin, C.J., 2019, <em>Essence in the Age of Evolution: A New
Theory of Natural Kinds</em>, New York: Routledge.</li>

<li>Bechtel, W, and R. Richardson, 1993, <em>Discovering Complexity:
Decomposition and Localization as Strategies in Scientific
Research</em>, Princeton: Princeton University Press.</li>

<li>Berrill, N.J., 1961, <em>Growth, Development, and Pattern</em>,
San Francisco: W.H. Freeman and Company.</li>

<li>Bier, E., and W. McGinnis, 2003, &ldquo;Model organisms in the
study of development and disease&rdquo;, in <em>Molecular Basis of
Inborn Errors of Development</em>, C.J. Epstein, R.P. Erickson, and A.
Wynshaw-Boris (eds.), 25&ndash;45. New York: Oxford University
Press.</li>

<li>Bolker, J.A., 1995, &ldquo;Model systems in developmental
biology&rdquo;, <em>BioEssays</em>, 17: 451&ndash;455.</li>

<li>&ndash;&ndash;&ndash;, 2009, &ldquo;Exemplary and surrogate
models: Two modes of representation in biology&rdquo;,
<em>Perspectives in Biology and Medicine</em>, 52: 485&ndash;499.</li>

<li>Brand, T., 2003, &ldquo;Heart development: molecular insights into
cardiac specification&rdquo;, <em>Developmental Biology</em>, 258:
1&ndash;19.</li>

<li>Brigandt, I., 2013, &ldquo;Systems biology and the integration of
mechanistic explanation and mathematical explanation&rdquo;,
<em>Studies in History and Philosophy of Biological and Biomedical
Sciences</em>, 44: 477&ndash;492.</li>

<li>Brigandt, I., and A.C. Love, 2012, &ldquo;Conceptualizing
evolutionary novelty: Moving beyond definitional debates&rdquo;,
<em>Journal of Experimental Zoology (Mol Dev Evol)</em>, 318B:
417&ndash;427.</li>

<li>Brouz&eacute;s, E., and E. Farge, 2004, &ldquo;Interplay of
mechanical deformation and patterned gene expression in developing
embryos&rdquo;, <em>Current Opinion in Genetics &amp;
Development</em>, 14: 367&ndash;374.</li>

<li>Colbert, M.W., and T. Rowe, 2008, &ldquo;Ontogenetic Sequence
Analysis: using parsimony to characterize developmental sequences and
sequence polymorphism&rdquo;, <em>Journal of Experimental Zoology (Mol
Dev Evol)</em>, 310B: 398&ndash;416.</li>

<li>Craver, C.F., 2007, <em>Explaining the Brain: Mechanisms and the
Mosaic Unity of Neuroscience</em>, New York: Oxford University
Press.</li>

<li>Crotty, D.A., and A. Gann, 2009, <em>Emerging Model Organisms: A
Laboratory Manual, Volume 1</em>, Cold Spring Harbor, NY: Cold Spring
Harbor Laboratory Press.</li>

<li>Damen, W.G.M., 2007, &ldquo;Evolutionary conservation and
divergence of the segmentation process in arthropods&rdquo;,
<em>Developmental Dynamics</em>, 236: 1379&ndash;1391.</li>

<li>Darden, L., 2006, <em>Reasoning in Biological Discoveries: Essays
on Mechanisms, Interfield Relations, and Anomaly Resolution</em>, New
York: Cambridge University Press.</li>

<li>Davidson, E.H., 2006, <em>The Regulatory Genome: Gene Regulatory
Networks in Development and Evolution</em>, San Diego: Academic
Press.</li>

<li>Davidson, E.H. and I.S. Peter, 2015, <em>Genomic Control Process:
Development and Evolution</em>, San Diego, CA: Academic Press.</li>

<li>Davies, J.A., 2013, <em>Mechanisms of Morphogenesis: The Creation
of Biological Form</em>, 2nd edition, San Diego, CA: Elsevier Academic
Press.</li>

<li>Davis, R.H., 2004, &ldquo;The age of model organisms&rdquo;,
<em>Nature Reviews Genetics</em>, 5: 69&ndash;76.</li>

<li>DeWitt, T.J., and S.M. Scheiner, 2004, <em>Phenotypic Plasticity:
Functional and Conceptual Approaches</em>, New York: Oxford University
Press.</li>

<li>DiTeresi, C.A., 2010, &ldquo;Taming Variation: Typological
Thinking and Scientific Practice in Developmental Biology&rdquo;,
University of Chicago, Chicago. PhD Thesis.</li>

<li>Dubuis, J.O., G. Tka&#269;ik, E.F. Wieschaus, T. Gregor and W.
Bialek, 2013, &ldquo;Positional information, in bits&rdquo;,
<em>Proceedings of the National Academy of Sciences</em>, 110:
16301&ndash;16308.</li>

<li>Dupr&eacute;, J., 2001, &ldquo;In defence of
classification&rdquo;, <em>Studies the History and Philosophy of
Biological and Biomedical Sciences</em>, 32: 203&ndash;219.</li>

<li>Ellis, R.J., 2001, &ldquo;Macromolecular crowding: obvious but
underappreciated&rdquo;, <em>Trends in Biochemical Sciences</em>, 26:
597&ndash;604.</li>

<li>Emlen, D.J., 2000, &ldquo;Integrating development with evolution:
A case study with beetle horns&rdquo;, <em>BioScience</em>, 50:
403&ndash;418.</li>

<li>Facchin, S., R. Lopreiato, M. Ruzzene, O. Marin, G. Sartori, C.
G&ouml;tz, M. Montenarh, G. Carignani, and L.A. Pinna, 2003,
&ldquo;Functional homology between yeast piD261/Bud32 and human PRPK:
both phosphorylate p53 and PRPK partially complements piD261/Bud32
deficiency&rdquo;, <em>FEBS Letters</em>, 549: 63&ndash;66.</li>

<li>Fagan, M.B., 2013, <em>Philosophy of Stem Cell Biology: Knowledge
in Flesh and Blood</em>, London: Palgrave Macmillan.</li>

<li>&ndash;&ndash;&ndash;, 2016, &ldquo;Generative models: human
embryonic stem cells and multiple modeling relations&rdquo;,
<em>Studies in History and Philosophy of Science</em>, 56:
122&ndash;134.</li>

<li>Forgacs, G., and S.A. Newman, 2005, <em>Biological Physics of the
Developing Embryo</em>, New York: Cambridge University Press.</li>

<li>Frankino, W.A., and R.A. Raff, 2004, &ldquo;Evolutionary
importance and pattern of phenotypic plasticity&rdquo;, in DeWitt and
Scheiner 2004: 64&ndash;81.</li>

<li>Fraser, S.E., and R.M. Harland, 2000, &ldquo;The molecular
metamorphosis of experimental embryology&rdquo;, <em>Cell</em>, 100:
41&ndash;55.</li>

<li>Furley, D., and J.S. Wilkie, 1984, <em>Galen: On Respiration and
the Arteries</em>, Princeton: Princeton University Press.</li>

<li>Gajewski, K., and R.A. Schulz, 2002, &ldquo;Comparative genetics
of heart development: conserved cardiogenic factors in
<em>Drosophila</em> and vertebrates&rdquo;, in <em>Cardiac
Development</em>, B. Ostadal, M. Nagano, and N.S. Dhalla (eds.),
1&ndash;23. Boston: Kluwer Academic Publishers.</li>

<li>Garson, J., 2016, <em>A Critical Overview of Biological
Functions</em>, Dordrecht: Springer.</li>

<li>Gerhart, J., and M. Kirschner, 2007, &ldquo;The theory of
facilitated variation&rdquo;, <em>Proceedings of the National Academy
of Sciences USA</em>, 104: 8582&ndash;8589.</li>

<li>Ghiselin, M.T., 2005, &ldquo;Homology as a relation of
correspondence between parts of individuals&rdquo;, <em>Theory in
Biosciences</em>, 124: 91&ndash;103.</li>

<li>Giere, R.N., 1999, <em>Science Without Laws</em>, Chicago:
University of Chicago Press.</li>

<li>Gilbert, S.F. (ed.), 1991, <em>A Conceptual History of Modern
Embryology</em> (Volume 7: <em>Developmental Biology: A Comprehensive
Synthesis</em>), New York: Plenum Press.</li>

<li>&ndash;&ndash;&ndash;, 2000 [2003, 2006, 2010], <em>Developmental
Biology</em>, 6th edition, Sunderland, MA: Sinauer Associates, Inc.,
2000; 7th edition, 2003; 8th edition, 2006; 9th edition, 2010.</li>

<li>Gilbert, S.F. and D. Epel, 2009, <em>Ecological Developmental
Biology: Integrating Epigenetics, Medicine, and Evolution</em>,
Sunderland, MA: Sinauer.</li>

<li>Glennan, S. and P. Illari (eds.), 2017, <em>The Routledge Handbook
of the Philosophy of Mechanisms and Mechanical Philosophy</em>, New
York: Routledge.</li>

<li>Green, S. and R. Batterman, 2017, &ldquo;Biology meets physics:
reductionism and multi-scale modeling of morphogenesis&rdquo;,
<em>Studies in History and Philosophy of Biological and Biomedical
Sciences</em>, 61: 20&ndash;34.</li>

<li>Greene, E., 1989, &ldquo;A diet-induced developmental polymorphism
in a caterpillar&rdquo;, <em>Science</em>, 243: 643&ndash;646.</li>

<li>Griesemer, J.R., 1996, &ldquo;Periodization and models in
historical biology&rdquo;, in <em>New Perspectives on the History of
Life</em>, M.T. Ghiselin, and G. Pinna (eds.), 19&ndash;30. San
Francisco: California Academy of Sciences.</li>

<li>Griffiths, P. and K. Stotz, 2013, <em>Genetics and Philosophy: An
Introduction</em>, New York: Cambridge University Press.</li>

<li>Gu, Z., L. David, D. Petrov, T. Jones, R.W. Davis, and L.M.
Steinmetz, 2005, &ldquo;Elevated evolutionary rates in the laboratory
strain of <em>Saccharomyces cerevisiae</em>&rdquo;, <em>Proceedings of
the National Academy of Sciences USA</em>, 102: 1092&ndash;1097.</li>

<li>Gulledge, A.T., and Y. Kawaguchi, 2007, &ldquo;Phasic cholinergic
signaling in the hippocampus: functional homology with the
neocortex?&rdquo; <em>Hippocampus</em>, 17: 327&ndash;332.</li>

<li>Hall, B.K. (ed.), 1994, <em>Homology: The Hierarchical Basis of
Comparative Biology</em>, San Diego: Academic Press.</li>

<li>Hall, L.R., 2005, &ldquo;Exploratory experiments&rdquo;,
<em>Philosophy of Science</em>, 72: 888&ndash;899.</li>

<li>Hamburger, V., 1988, <em>The Heritage of Experimental Embryology:
Hans Spemann and the Organizer</em>, New York: Oxford University
Press.</li>

<li>Hamburger, V., and H.L. Hamilton, 1951, &ldquo;A series of normal
stages in the development of the chick embryo&rdquo;, <em>Journal of
Morphology</em>, 88: 49&ndash;92.</li>

<li>Harvey, R.P., 2002, &ldquo;Patterning the vertebrate heart&rdquo;,
<em>Nature Reviews Genetics</em>, 3: 544&ndash;556.</li>

<li>Hoffman, B.D., C. Grashoff, and M.A. Schwartz, 2011,
&ldquo;Dynamic molecular processes mediate cellular
mechanotransduction&rdquo;, <em>Nature</em>, 475: 316&ndash;323.</li>

<li>Hopwood, N., 1999, &ldquo;&lsquo;Giving body&rsquo; to embryos:
modeling, mechanism, and the microtome in late nineteenth-century
anatomy&rdquo;, <em>Isis</em>, 90: 462&ndash;496.</li>

<li>&ndash;&ndash;&ndash;, 2000, &ldquo;Producing development: the
anatomy of human embryos and the norms of Wilhelm His&rdquo;,
<em>Bulletin of the History of Medicine</em>, 74: 29&ndash;79.</li>

<li>&ndash;&ndash;&ndash;, 2005, &ldquo;Visual standards and
disciplinary change: normal plates, tables and stages in
embryology&rdquo;, <em>History of Science</em>, 43:
239&ndash;303.</li>

<li>&ndash;&ndash;&ndash;, 2007, &ldquo;A history of normal plates,
tables and stages in vertebrate embryology&rdquo;, <em>International
Journal of Developmental Biology</em>, 51: 1&ndash;26.</li>

<li>&ndash;&ndash;&ndash;, 2019, &ldquo;Inclusion and exclusion in the
history of developmental biology&rdquo;, <em>Development</em>, 146(7):
dev175448. doi:10.1242/dev.175448</li>

<li>Horder, T.J., J.A. Witkowski, and C.C. Wylie (eds), 1986, <em>A
History of Embryology</em>, Cambridge: Cambridge University
Press.</li>

<li>Hove, J.R., R.W. K&ouml;ster, A.S. Forouhar, G. Acevedo-Bolton,
S.E. Fraser, and M. Gharib, 2003, &ldquo;Intracardiac fluid forces are
an essential epigenetic factor for embryonic cardiogenesis&rdquo;,
<em>Nature</em>, 421: 172&ndash;177.</li>

<li>Huang, A., C.A. Scougall, J.W. Lowenthal, A.R. Jilbert, and I.
Kotlarski, 2001, &ldquo;Structural and functional homology between
duck and chicken interferon-gamma&rdquo;, <em>Developmental and
Comparative Immunology</em>, 25: 55&ndash;68.</li>

<li>H&uuml;ttemann, A. and M.I. Kaiser., 2018, &ldquo;Potentiality in
biology&rdquo;, in <em>Handbook of Potentiality</em>, K. Engelhardt
and M. Quante (eds.), 401&ndash;428. Dordrecht: Springer.</li>

<li>Illari, P., and J. Williamson, 2012, &ldquo;What is a mechanism?
Thinking about mechanisms across the sciences&rdquo;, <em>European
Journal of the Philosophy of Science</em>, 2: 119&ndash;135.</li>

<li>Jackson, T.R., H.Y. Kim, U.L. Balakrishnan, C. Stuckenholz and
L.A. Davidson, 2017, &ldquo;Spatiotemporally controlled mechanical
cues drive progenitor mesenchymal-to-epithelial transition enabling
proper heart formation and function&rdquo;, <em>Current Biology</em>,
27: 1326&ndash;1335.</li>

<li>Jones, M.R., 2005, &ldquo;Idealization and abstraction: a
framework&rdquo;, in <em>Idealization XII: Correcting the Model.
Idealization and Abstraction in the Sciences (Poznan Studies in the
Philosophy of the Sciences and the Humanities, vol. 86)</em>, M.R.
Jones and N. Cartwright (eds.), 173&ndash;217. Amsterdam/New York:
Rodopi.</li>

<li>Kaplan, J.M., 2008, &ldquo;Phenotypic plasticity and reaction
norms&rdquo;, in Sarkar and Plutynski 2008: 205&ndash;222.</li>

<li>Keller, E.F., 2002, <em>Making Sense of Life: Explaining
Biological Development with Models, Metaphors, and Machines</em>,
Cambridge, MA: Harvard University Press.</li>

<li>Kimmel, C.B., W.W. Ballard, S.R. Kimmel, B. Ullmann, and T.F.
Schilling, 1995, &ldquo;Stages of embryonic development of the
zebrafish&rdquo;, <em>Developmental dynamics</em>, 203:
253&ndash;310.</li>

<li>Kingma, E., 2018, &ldquo;Lady parts: the metaphysics of
pregnancy&rdquo;, <em>Royal Institute of Philosophy Supplement</em>,
82: 165&ndash;187.</li>

<li>Kirby, M.L., 1999, &ldquo;Contribution of neural crest to heart
and vessel morphology&rdquo;, in <em>Heart Development</em>, R.P.
Harvey and N. Rosenthal (eds.), 179&ndash;193. San Diego: Academic
Publishers.</li>

<li>Kirschner, M.W., and J.C. Gerhart, 2005, <em>The Plausibility of
Life: Resolving Darwin&rsquo;s Dilemma</em>, New Haven and London:
Yale University Press.</li>

<li>Kitcher, P., 1993, <em>The Advancement of Science: Science Without
Legend, Objectivity Without Illusions</em>, New York: Oxford
University Press.</li>

<li>Laplane, L., 2016, <em>Cancer Stem Cells: Philosophy and
Therapies</em>, Cambridge, MA: Harvard University Press.</li>

<li>Levin, M., T. Thorlin, K.R. Robinson, T. Nogi, and M. Mercola,
2002, &ldquo;Asymmetries in H<sup>+</sup>/K<sup>+</sup> -ATPase and
cell membrane potentials comprise a very early step in left-right
patterning&rdquo;, <em>Cell</em>, 111: 77&ndash;89.</li>

<li>Love, A.C., 2007, &ldquo;Functional homology and homology of
function: biological concepts and philosophical consequences&rdquo;,
<em>Biology &amp; Philosophy</em>, 22: 691&ndash;708.</li>

<li>&ndash;&ndash;&ndash;, 2008, &ldquo;Explaining the ontogeny of
form: philosophical issues&rdquo;, in Sarkar and Plutynski 2008:
223&ndash;247.</li>

<li>&ndash;&ndash;&ndash;, 2010, &ldquo;Idealization in evolutionary
developmental investigation: a tension between phenotypic plasticity
and normal stages&rdquo;, <em>Philosophical Transactions of the Royal
Society B: Biological Sciences</em>, 365: 679&ndash;690.</li>

<li>&ndash;&ndash;&ndash;, 2012, &ldquo;Formal and material theories
in philosophy of science: A methodological interpretation&rdquo;, in
<em>EPSA Philosophy of Science: Amsterdam 2009 (The European
Philosophy of Science Association Proceedings, Vol. 1)</em>, H.W. de
Regt, S. Okasha, and S. Hartmann (eds.), 175&ndash;185. Berlin:
Springer.</li>

<li>&ndash;&ndash;&ndash;, 2013, &ldquo;Theory is as theory does:
Scientific practice and theory structure in biology&rdquo;,
<em>Biological Theory</em>, 7: 325&ndash;337.</li>

<li>&ndash;&ndash;&ndash;, 2014, &ldquo;The erotetic organization of
development&rdquo;, in <em>Towards a Theory of Development</em>, A.
Minelli, and T. Pradeu (eds.), 33&ndash;55. Oxford: Oxford University
Press.</li>

<li>&ndash;&ndash;&ndash;, 2015, &ldquo;Evolutionary developmental
biology: philosophical issues&rdquo;, in <em>Handbook of Evolutionary
Thinking in the Sciences</em>, T. Heams, P. Huneman, L. Lecointre, and
M. Silberstein (eds.), 265&ndash;283, Berlin: Springer.</li>

<li>&ndash;&ndash;&ndash;, 2017a, &ldquo;Developmental
mechanisms&rdquo;, in <em>The Routledge Handbook of the Philosophy of
Mechanisms and Mechanical Philosophy</em>, S. Glennan and P. Illari
(eds.), 332&ndash;347, New York: Routledge.</li>

<li>&ndash;&ndash;&ndash;, 2017b, &ldquo;Building integrated
explanatory models of complex biological phenomena: from Mill&rsquo;s
methods to a causal mosaic&rdquo;, in <em>EPSA15 Selected Papers: The
5th conference of the European Philosophy of Science Association</em>,
M. Massimi, J.-W. Romeijn and G. Schurz (eds.), 221&ndash;232, Cham:
Springer International Publishing.</li>

<li>Love, A.C., and M. Travisano, 2013, &ldquo;Microbes modeling
ontogeny&rdquo;, <em>Biology &amp; Philosophy</em>, 28:
161&ndash;188.</li>

<li>Lowe, J.W.E., 2015, &ldquo;Managing variation in the investigation
of organismal development: problems and opportunities&rdquo;,
<em>History and Philosophy of the Life Sciences</em>, 37:
449&ndash;473.</li>

<li>&ndash;&ndash;&ndash;, 2016, &ldquo;Normal development and
experimental embryology: Edmund Beecher Wilson and
<i>Amphioxus</i>&rdquo;, <em>Studies in History and Philosophy of
Biological and Biomedical Sciences</em>, 57: 44&ndash;59.</li>

<li>Lynch, V.J., 2009, &ldquo;Use with caution: developmental systems
divergence and potential pitfalls of animal models&rdquo;, <em>Yale
Journal of Biology and Medicine</em>, 82: 53&ndash;66.</li>

<li>Mabee, P.M., K.L. Olmstead, and C.C. Cubbage, 2000, &ldquo;An
experimental study of intrspecific variation, developmental timing,
and heterochrony in fishes&rdquo;, <em>Evolution</em>, 54:
2091&ndash;2106.</li>

<li>Maienschein, J., 1991, <em>Transforming Traditions in American
Biology, 1880&ndash;1915</em>, Baltimore, MD: The Johns Hopkins
University Press.</li>

<li>&ndash;&ndash;&ndash;, 2000, &ldquo;Competing epistemologies and
developmental biology&rdquo;, in <em>Biology and Epistemology</em>, R.
Creath and J. Maienschein (eds.), 122&ndash;137. Cambridge: Cambridge
University Press.</li>

<li>&ndash;&ndash;&ndash;, 2014, <em>Embryos under the Microscope: The
Diverging Meanings of Life </em>, Cambridge, MA: Harvard University
Press.</li>

<li>Maienschein, J., M. Glitz, and G.E. Allen, 2005, <em>Centennial
History of the Carnegie Institution of Washington: Volume 5, The
Department of Embryology</em>, New York: Cambridge University
Press.</li>

<li>Manak, J.R., and M.P. Scott, 1994, &ldquo;A class act:
conservation of homeodomain protein functions&rdquo;,
<em>Development</em>, (Supplement): 61&ndash;71.</li>

<li>McManus, F., 2012, &ldquo;Development and mechanistic
explanation&rdquo;, <em>Studies in History and Philosophy of
Biological and Biomedical Sciences</em>, 43: 532&ndash;541.</li>

<li>Metscher, B.D., and P.E. Ahlberg, 1999, &ldquo;Zebrafish in
context: uses of a laboratory model in comparative studies&rdquo;,
<em>Developmental Biology</em>, 210: 1&ndash;14.</li>

<li>Mill, J.S., 1843 [1974], <em>A System of Logic Ratiocinative and
Inductive, Being a Connected View of the Principles of Evidence and
the Methods of Scientific Investigation (Books I&ndash;III)</em>, in
<em>The Collected Works of John Stuart Mill, Volume VII</em>, John M.
Robson (ed.), Toronto: University of Toronto Press, London: Routledge
and Kegan Paul.
 [<a href="http://oll.libertyfund.org/titles/246" target="other">Mill 1843 [1974] available online</a>]</li>
 
<li>Miller, C.J., and L.A. Davidson, 2013, &ldquo;The interplay
between cell signalling and mechanics in developmntal
processes&rdquo;, <em>Nature Reviews Genetics</em>, 14:
733&ndash;744.</li>

<li>Minelli, A., 2003, <em>The Development of Animal Form: Ontogeny,
Morphology, and Evolution</em>, Cambridge: Cambridge University
Press.</li>

<li>&ndash;&ndash;&ndash;, 2011a, &ldquo;Animal development, an
open-ended segment of life&rdquo;, <em>Biological Theory</em>, 6:
4&ndash;15.</li>

<li>&ndash;&ndash;&ndash;, 2011b, &ldquo;A principle of developmental
inertia&rdquo;, in <em>Epigenetics: Linking Genotype and Phenotype in
Development and Evolution</em>, B. Halgr&iacute;msson and B.K. Hall
(eds.), 116&ndash;133. San Francisco: University of California
Press.</li>

<li>Minelli, A., C. Brena, G. Deflorian, D. Maruzzo, and G. Fusco,
2006, &ldquo;From embryo to adult-beyond the conventional
periodization of arthropod development&rdquo;, <em>Development Genes
and Evolution</em>, 216: 373&ndash;383.</li>

<li>Minelli, A. and T. Pradeu (eds.), 2014, <em>Towards a Theory of
Development</em>, Oxford: Oxford University Press.</li>

<li>Mitchell, S.D., 2002, &ldquo;Integrative pluralism&rdquo;,
<em>Biology &amp; Philosophy</em>, 17: 55&ndash;70.</li>

<li>Moczek, A., 2008, &ldquo;On the origins of novelty in development
and evolution&rdquo;, <em>BioEssays</em>, 30: 432&ndash;447.</li>

<li>Moczek, A.P., and L.M. Nagy, 2005, &ldquo;Diverse developmental
mechanisms contribute to different levels of diversity in horned
beetles&rdquo;, <em>Evolution &amp; Development</em>, 7:
175&ndash;185.</li>

<li>Moczek, A.P., S. Sultan, S. Foster, C. Led&oacute;n-Rettig, I.
Dworkin, H.F. Nijhout, E. Abouheif and D.W. Pfennig, 2011, &ldquo;The
role of developmental plasticity in evolutionary innovation&rdquo;,
<em>Proceedings of the Royal Society of London B: Biological
Sciences</em>, 278: 2705&ndash;2713.</li>

<li>Morgan, T.H., 1923, &ldquo;The modern theory of genetics and the
problem of embryonic development&rdquo;, <em>Physiological
Review</em>, 3: 603&ndash;627.</li>

<li>&ndash;&ndash;&ndash;, 1926, &ldquo;Genetics and the physiology of
development&rdquo;, <em>American Naturalist</em>, 60:
489&ndash;515.</li>

<li>&ndash;&ndash;&ndash;, 1934, <em>Embryology and Genetics</em>, New
York: Columbia University Press.</li>

<li>Moss, L., 2002, <em>What Genes Can&rsquo;t Do</em>, Cambridge, MA:
MIT Press, A Bradford Book.</li>

<li>M&uuml;ller, G.B., 2007, &ldquo;Evo-devo: extending the
evolutionary synthesis&rdquo;, <em>Nature Reviews Genetics</em>, 8:
943&ndash;949.</li>

<li>Nagel, E., 1961, <em>The Structure of Science: Problems in the
Logic of Scientific Explanation</em>, New York: Harcourt, Brace &amp;
World, Inc.</li>

<li>Neumann-Held, E.M., and C. Rehmann-Sutter (eds.), 2006, <em>Genes
in Development: Re-reading the Molecular Paradigm</em>, Durham and
London: Duke University Press.</li>

<li>Newman, S.A., 2015, &ldquo;Development and evolution: The physics
connection&rdquo;, in <em>Conceptual Change in Biology: Scientific and
Philosophical Perspectives on Evolution and Development</em>, A.C.
Love (ed.), 421&ndash;440, Berlin: Springer.</li>

<li>Newman, S.A., and R. Bhat, 2008, &ldquo;Dynamical patterning
modules: physico-genetic determinants of morphological development and
evolution&rdquo;, <em>Physical Biology</em>, 5: 1&ndash;14.</li>

<li>Nonaka, S., H. Shiratori, Y. Saijoh, and H. Hamada, 2002,
&ldquo;Determination of left-right patterning of the mouse embryo by
artificial nodal flow&rdquo;, <em>Nature</em>, 418: 96&ndash;99.</li>

<li>N&uuml;sslein-Volhard, C., 2006, <em>Coming to Life: How Genes
Drive Development</em>, Carlsbad, CA: Kales Press.</li>

<li>Olby, R.C., 1986, &ldquo;Structural and dynamical explanations in
the world of neglected dimensions&rdquo;, in Horder, Witkowski, and
Wylie 1986: 275&ndash;308.</li>

<li>Olson, E.N., 2006, &ldquo;Gene regulatory networks in the
evolution and development of the heart&rdquo;, <em>Science</em>, 313:
1922&ndash;1927.</li>

<li>O&rsquo;Malley, M.A., 2007, &ldquo;Exploratory experimentation and
scientific practice: metagenomics and the proteorhodopsin case&rdquo;,
<em>History and Philosophy of the Life Sciences</em>, 29:
337&ndash;60.</li>

<li>Oppenheimer, J.M., 1967, <em>Essays in the History of Embryology
and Biology</em>, Cambridge, MA: MIT Press.</li>

<li>Overton, J.A., 2013, &ldquo;&lsquo;Explain&rsquo; in scientific
discourse&rdquo;, <em>Synthese</em>, 190: 1383&ndash;1405.</li>

<li>Owen, R., 1843, <em>Lectures on the Comparative Anatomy and
Physiology of the Invertebrate Animals</em>, London: Longman, Brown,
Green, and Longmans.</li>

<li>Pabst, D.A., 2000, &ldquo;To bend a dolphin: convergence of force
transmission designs in Cetaceans and Scombrid fishes&rdquo;,
<em>American Zoologist</em>, 40: 146&ndash;155.</li>

<li>Parkkinen, V-P., 2014, &ldquo;Developmental explanations&rdquo;,
in <em>New Directions in the Philosophy of Science: The Philosophy of
Science in a European Perspective, Vol. 5</em>, M.C. Galavotti, D.
Dieks, W.J. Gonzalez, S. Hartmann, T. Uebel, and M. Weber (eds.),
Berlin: Springer.</li>

<li>Pearson, C., 2018, &ldquo;How-possibly explanation in biology:
lessons from Wilhelm His&rsquo;s &lsquo;simple experiments&rsquo;
models&rdquo;, <em>Philosophy, Theory, and Practice in Biology</em>,
10(4), doi:10.3998/ptpbio.16039257.0010.004</li>

<li>Peter, I.S., and E.H. Davidson, 2011, &ldquo;A gene regulatory
network controlling the embryonic specification of endoderm&rdquo;,
<em>Nature</em>, 474: 635&ndash;639.</li>

<li>Pigliucci, M., 2001, <em>Phenotypic Plasticity: Beyond Nature and
Nurture</em>, Baltimore and London: The Johns Hopkins University
Press.</li>

<li>&ndash;&ndash;&ndash;, 2002, &ldquo;Touchy and bushy: phenotypic
plasticity and integration in response to wind stimulation in
<em>Arabidopsis thaliana</em>&rdquo;, <em>International Journal of
Plant Sciences</em>, 163: 399&ndash;408.</li>

<li>Raff, R.A., 2000, &ldquo;Evo-Devo: the evolution of a new
discipline&rdquo;, <em>Nature Reviews Genetics</em>, 1:
74&ndash;79.</li>

<li>Ranganayakulu, G., D.A. Elliott, R.P. Harvey, and E.N. Olson,
1998, &ldquo;Divergent roles for <em>NK-2</em> class homeobox genes in
cardiogenesis in flies and mice&rdquo;, <em>Development</em>, 125:
3037&ndash;3048.</li>

<li>Raya, &Aacute;., Y. Kawakami, C. Rodr&iacute;guez-Esteban, M.
Iba&ntilde;es, D. Rasskin-Gutman, J. Rodr&iacute;guez-Le&oacute;n, D.
B&uuml;scher, J.A. Feij&oacute;, and J.C.I. Belmonte, 2004,
&ldquo;Notch activity acts as a sensor for extracellular calcium
during vertebrate left-right determination&rdquo;, <em>Nature</em>,
427: 121&ndash;128.</li>

<li>Reed, R.D., P-H. Chen, and H.F Nijhout, 2007, &ldquo;Cryptic
variation in butterfly eyespot development: the importance of sample
size in gene expression studies&rdquo;, <em>Evolution &amp;
Development</em>, 9: 2&ndash;9.</li>

<li>Reiss, J.O., 2003, &ldquo;Time&rdquo;, in <em>Keywords and
Concepts in Evolutionary Developmental Biology</em>, B.K. Hall, and
W.M. Olson (eds.), 359&ndash;368. Cambridge, MA: Harvard University
Press.</li>

<li>Robert, J.S., 2004, <em>Embryology, Epigenesis, and Evolution:
Taking Development Seriously</em>, New York: Cambridge University
Press.</li>

<li>Roe, S.A., 1981, <em>Matter, Life, and Generation: 18th Century
Embryology and the Haller-Wolff Debate</em>, Cambridge: Cambridge
University Press.</li>

<li>Rosenberg, A., 2006, <em>Darwinian Reductionism: Or, How to Stop
Worrying and Love Molecular Biology</em>, Chicago: University of
Chicago Press.</li>

<li>Rubin, G.M., 1988, <em>Drosophila melanogaster</em> as an
experimental organism. <em>Science</em>, 240: 1453&ndash;1459.</li>

<li>Sarkar, S. and A. Plutynski (eds.), 2008, <em>A Companion to the
Philosophy of Biology</em>, (Blackwell Companions to Philosophy),
Malden, MA: Blackwell Publishers.</li>

<li>Savin, T., N.A. Kurpios, A.E. Shyer, P. Florescu, H. Liang, L.
Mahadevan, and C. Tabin, 2011, &ldquo;On the growth and form of the
gut&rdquo;, <em>Nature</em>, 476: 57&ndash;62.</li>

<li>Scheiner, S.M., 1993, &ldquo;Genetics and evolution of phenotypic
plasticity&rdquo;, <em>Annual Review of Ecology and Systematics</em>,
24: 35&ndash;68.</li>

<li>Schulze, J., and E. Schierenberg, 2011, &ldquo;Evolution of
embryonic development in nematodes&rdquo;, <em>EvoDevo</em>, 2:
18.</li>

<li>Sheil, C.A., and E. Greenbaum, 2005, &ldquo;Reconsideration of
skeletal development of <em>Chelydra serpentina</em> (Reptilia:
Testudinata: Chelydridae): evidence for intraspecific
variation&rdquo;, <em>Journal of Zoology</em>, 265:
235&ndash;267.</li>

<li>Sidzinska, M., 2017, &ldquo;Not one, not two: toward an ontology
of pregnancy&rdquo;, <em>Feminist Philosophy Quarterly</em>, 3(4),
Article 2, doi:10.5206/fpq/2017.4.2</li>

<li>Slack, J.M.W., 2006, <em>Essential Developmental Biology</em>, 2nd
ed. Malden, MA: Blackwell Publishing.</li>

<li>&ndash;&ndash;&ndash;, 2009, &ldquo;Emerging market
organisms&rdquo;, <em>Science</em>, 323: 1674&ndash;1675.</li>

<li>&ndash;&ndash;&ndash;, 2013, <em>Essential Developmental
Biology</em>, 3rd edition, Chichester: Wiley-Blackwell.</li>

<li>Smith, J.E.H. (ed.), 2006, <em>The Problem of Animal Generation in
Early Modern Philosophy</em>, New York: Cambridge University
Press.</li>

<li>&ndash;&ndash;&ndash;, 2011, <em>Divine Machines: Leibniz and the
Sciences of Life</em>, Princeton, NJ: Princeton University Press.</li>

<li>Sober, E., 1988, &ldquo;Apportioning causal responsibility&rdquo;,
<em>Journal of Philosophy</em>, 85: 303&ndash;318.</li>

<li>Sommer, R.J., 2009, &ldquo;The future of evo-devo: model systems
and evolutionary theory&rdquo;, <em>Nature Reviews Genetics</em>, 10:
416&ndash;422.</li>

<li>Srivastava, D., 2006, &ldquo;Making or breaking the heart: from
lineage determination to morphogenesis&rdquo;, <em>Cell</em>, 126:
1037&ndash;1048.</li>

<li>Steel, D.P., 2008, <em>Across the Boundaries: Extrapolation in
Biology and Social Science</em>, New York: Oxford University
Press.</li>

<li>Strevens, M., 2009, <em>Depth: An Account of Scientific
Explanation</em>, Cambridge, MA: Harvard University Press.</li>

<li>Thompson, D&rsquo;A.W. 1992 [1942]. <em>On Growth and Form</em>,
Complete Revised Edition. New York: Dover Publications, Inc.</li>

<li>Varner, V.D., and C.M. Nelson, 2014, &ldquo;Cellular and physical
mechanisms of branching morphogenesis&rdquo;, <em>Development</em>,
141: 2750&ndash;2759.</li>

<li>V&aacute;zquez-Novelle, M.D., V. Esteban, A. Bueno, and M.P.
Sacrist&aacute;n., 2005, &ldquo;Functional homology among human and
fission yeast Cdc14 phosphatases&rdquo;, <em>Journal of Biological
Chemistry</em>, 280: 29144&ndash;29150.</li>

<li>Von Dassow, M., J. Strother, and L.A. Davidson, 2010,
&ldquo;Surprisingly simple mechanical behavior of a complex embryonic
tissue&rdquo;, <em>PLoS ONE</em>, 5:e15359.</li>

<li>Waters, C.K., 2007a, &ldquo;Causes that make a difference&rdquo;,
<em>Journal of Philosophy</em>, 104: 551&ndash;579.</li>

<li>&ndash;&ndash;&ndash;, 2007b, &ldquo;The nature and context of
exploratory experimentation&rdquo;, <em>History and Philosophy of the
Life Sciences</em>, 29: 275&ndash;284.</li>

<li>Weber, M., 2005, <em>Philosophy of Experimental Biology</em>, New
York: Cambridge University Press.</li>

<li>Weisberg, M., 2007, &ldquo;Three kinds of idealization&rdquo;,
<em>Journal of Philosophy</em>, 104: 639&ndash;659.</li>

<li>West-Eberhard, M.J., 2003, <em>Developmental Plasticity and
Evolution</em>, New York: Oxford University Press.</li>

<li>Wimsatt, W.C., 1980, &ldquo;Reductionistic research strategies and
their biases in the units of selection controversy&rdquo;, in
<em>Scientific Discovery: Case Studies</em>, T. Nickles (ed.),
213&ndash;259. Dordrecht: D. Reidel Publishing Company.</li>

<li>Wolpert, L., R. Beddington, J. Brockes, T. Jessell, P.A. Lawrence,
and E.M. Meyerowitz, 1998, <em>Principles of Development</em>, New
York: Oxford University Press.</li>

<li>Wolpert, L., C. Tickle, T. Jessell, P. Lawrence, E. Meyerowitz, E.
Robertson, and J. Smith, 2010, <em>Principles of Development</em>, 4th
ed. New York and Oxford: Oxford University Press.</li>

<li>Woodward, J., 2003, <em>Making Things Happen: A Theory of Causal
Explanation</em>, New York: Oxford University Press.</li>

<li>Wouters, A., 2003, &ldquo;Four notions of biological
function&rdquo;, <em>Studies in the History and Philosophy of
Biological and Biomedical Sciences</em>, 34: 633&ndash;668.</li>

<li>&ndash;&ndash;&ndash;, 2005, &ldquo;The function debate in
philosophy&rdquo;, <em>Acta Biotheoretica</em>, 53:
123&ndash;151.</li>

<li>Wozniak, M., and C.S. Chen, 2009, &ldquo;Mechanotransduction in
development: a growing role for contractility&rdquo;, <em>Nature
Reviews Molecular Cell Biology</em>, 10: 34&ndash;43.</li>

<li>Yoshida, Y., forthcoming, &ldquo;Multiple-models juxtaposition and
trade-offs among modeling desiderata&rdquo;, <em>Philosophy of
Science</em>.</li>
</ul>

<h3><a id="FigCre">Figure Credits</a></h3>

<ul class="hanging">

 <li><a href="#fig1">Figure 1</a>:
 &ldquo;Preformation&rdquo;, drawn by Nicolaas Hartsoeker (<em>Essai
de Dioptrique</em>, 1694). Licensed under Public domain via Wikimedia
Commons.
 <a href="../../../commons.wikimedia.org/wiki/File_Preformation.html" target="other">http://commons.wikimedia.org/wiki/File:Preformation.GIF</a></li>
 
 <li><a href="#fig2">Figure 2</a>:
 &ldquo;Spiral cleavage in gastropod <em>Trochus</em>&rdquo; by Morgan
Q. Goulding, 2009, &ldquo;Cell Lineage of the Ilyanassa Embryo:
Evolutionary Acceleration of Regional Differentiation during Early
Development&ldquo;, <em>PLoS ONE</em>, 4(5): e5506,
doi:10.1371/journal.pone.0005506, Figure 1 TIFF. Licensed under
Creative Commons Attribution 2.5 via Wikimedia Commons.
 <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0005506" target="other">http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0005506</a>
 OR
 <a href="../../../commons.wikimedia.org/wiki/File_Spiral_cleavage_in_Trochus.html" target="other">http://commons.wikimedia.org/wiki/File:Spiral_cleavage_in_Trochus.png</a></li>
 
 <li><a href="#fig3">Figure 3</a>:
 &ldquo;Hematopoiesis simple&rdquo; by Mikael H&auml;ggstr&ouml;m (no
attribution required), from original by A. Rad (requires attribution)
- Image:Hematopoiesis_(human)_diagram.png by A. Rad. Licensed under
Creative Commons Attribution-Share Alike 3.0 via Wikimedia Commons.
 <a href="http://commons.wikimedia.org/wiki/File:Hematopoiesis_simple.svg" target="other">http://commons.wikimedia.org/wiki/File:Hematopoiesis_simple.svg</a></li>
 
<li>
 <a href="#fig4">Figure 4</a>:
 &ldquo;Wingless and Hedgehog reciprocal signaling during segmentation
of <em>Drosophila</em> embryos&rdquo; by Fred the Oyster (requires
attribution). Licensed under Creative Commons Attribution-Share Alike
4.0 via Wikimedia Commons.
 <a href="https://upload.wikimedia.org/wikipedia/commons/7/7c/Wingless_and_Hedgehog_reciprocal_signaling_during_segmentation_of_Drosophila_embryos.svg" target="other">https://upload.wikimedia.org/wikipedia/commons/7/7c/Wingless_and_Hedgehog_reciprocal_signaling_during_segmentation_of_Drosophila_embryos.svg</a></li>
 
<li>
 <a href="#fig5">Figure 5</a>:
 &ldquo;<em>Drosophila melanogaster</em>&mdash;side (aka)&rdquo; by
Andr&eacute; Karwath aka Aka - Own work. Licensed under Creative
Commons Attribution-Share Alike 2.5 via Wikimedia Commons:
 <a href="../../../commons.wikimedia.org/wiki/File_Drosophila_melanogaster_-_side_(aka).html" target="other">http://commons.wikimedia.org/wiki/File:Drosophila_melanogaster_-_side_(aka).jpg</a></li>
 
<li>
 <a href="#fig6">Figure 6</a>:
 &ldquo;Homology&rdquo; from George John Romanes, 1892 [1910],
<em>Darwin and after Darwin</em>, (fourth edition), Chicago: The Open
Court Publishing Company, Figure 5, Chapter 3, p. 56, &ldquo;Wings of
Reptile, Mammal, and Bird. Drawn from nature (Brit. Mus.)&rdquo;.
 <a href="../../../www.talkorigins.org/faqs/precursors/images/homology.jpg" target="other">http://www.talkorigins.org/faqs/precursors/images/homology.jpg</a>.
 Licensed under Public domain via Wikimedia Commons -
 <a href="../../../commons.wikimedia.org/wiki/File_Homology.html" target="other">http://commons.wikimedia.org/wiki/File:Homology.jpg</a></li>
 
<li>
 <a href="#fig7">Figure 7</a>:
 &ldquo;<em>Biston betularia</em> caterpillars on birch (left) and
willow (right)&rdquo; by Mohamed A.F. Noor, Robin S. Parnell, Bruce S.
Grant, 2008, &ldquo;A Reversible Color Polyphenism in American
Peppered Moth (<em>Biston betularia cognataria</em>)
Caterpillars&rdquo;, <em>PLoS ONE</em>, 3(9): e3142.
doi:10.1371/journal.pone.0003142.
 <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0003142" target="other">http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0003142</a>
 Licensed under Creative Commons Attribution 2.5 via Wikimedia Commons
-
 <a href="../../../commons.wikimedia.org/wiki/File_Biston_betularia.html" target="other">http://commons.wikimedia.org/wiki/File:Biston_betularia.png</a></li>
 </ul>
</div> 
<div id="academic-tools">

<h2 id="Aca">Academic Tools</h2>

<blockquote>
<table class="vert-top">
<tr>
<td><img src="../../symbols/sepman-icon.jpg" alt="sep man icon" /></td>
<td><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/archinfo.cgi?entry=biology-developmental" target="other">How to cite this entry</a>.</td>
</tr>

<tr>
<td><img src="../../symbols/sepman-icon.jpg" alt="sep man icon" /></td>
<td><a href="https://leibniz.stanford.edu/friends/preview/biology-developmental/" target="other">Preview the PDF version of this entry</a> at the
 <a href="https://leibniz.stanford.edu/friends/" target="other">Friends of the SEP Society</a>.</td>
</tr>

<tr>
<td><img src="../../symbols/inpho.png" alt="inpho icon" /></td>
<td><a href="https://www.inphoproject.org/entity?sep=biology-developmental&amp;redirect=True" target="other">Look up topics and thinkers related to this entry</a>
 at the Internet Philosophy Ontology Project (InPhO).</td>
</tr>

<tr>
<td><img src="../../symbols/pp.gif" alt="phil papers icon" /></td>
<td><a href="https://philpapers.org/sep/biology-developmental/" target="other">Enhanced bibliography for this entry</a>
at <a href="https://philpapers.org/" target="other">PhilPapers</a>, with links to its database.</td>
</tr>

</table>
</blockquote>
</div>

<div id="other-internet-resources">

<h2><a id="Oth">Other Internet Resources</a></h2>

<ul>

 <li><a href="http://embryo.asu.edu/" target="other">Embryo Project</a></li>
 
 <li><a href="http://www.visembryo.com/" target="other">Visible Embryo (Human Development)</a></li>
 
 <li><a href="http://www.sdbonline.org/" target="other">Society for Developmental Biology</a></li>
 
 <li><a href="http://www.nih.gov/science/models/" target="other">Model Organisms for Biological Research</a></li>
 
 <li><a href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)1097-0177/homepage/custom_designed_page_a3.htm" target="other">Normal Stages for Model Organisms Used in Developmental Biology</a></li>
 </ul>
</div>

<div id="related-entries">

<h2><a id="Rel">Related Entries</a></h2>

<p>

 <a href="../aristotle-biology/index.html">Aristotle, General Topics: biology</a> |
 <a href="../aristotle-metaphysics/index.html">Aristotle, General Topics: metaphysics</a> |
 <a href="../theories-biological-development/index.html">biological development: theories of</a> |
 <a href="../biology-experiment/index.html">biology: experiment in</a> |
 <a href="../biology-philosophy/index.html">biology: philosophy of</a> |
 <a href="../boyle/index.html">Boyle, Robert</a> |
 <a href="../causation-mani/index.html">causation: and manipulability</a> |
 <a href="../cell-biology/index.html">cell biology, philosophy of</a> |
 <a href="../conway/index.html">Conway, Lady Anne</a> |
 <a href="../descartes/index.html">Descartes, Ren&eacute;</a> |
 <a href="../dispositions/index.html">dispositions</a> |
 <a href="../epistemology/index.html">epistemology</a> |
 <a href="../evolution/index.html">evolution</a> |
 <a href="../evolution-before-darwin/index.html">evolution: concept before Darwin</a> |
 <a href="../origin-descent/index.html">evolution: from <em>Origin of Species</em> to the <em>Descent of Man</em></a> |
 <a href="../gassendi/index.html">Gassendi, Pierre</a> |
 <a href="../gene/index.html">gene</a> |
 <a href="../genetics/index.html">genetics</a> |
 <a href="../genotype-phenotype/index.html">genetics: genotype/phenotype distinction</a> |
 <a href="../information-biological/index.html">information: biological</a> |
 <a href="../kant/index.html">Kant, Immanuel</a> |
 <a href="../leibniz/index.html">Leibniz, Gottfried Wilhelm</a> |
 <a href="../levels-org-biology/index.html">levels of organization in biology</a> |
 <a href="../life/index.html">life</a> |
 <a href="../malebranche/index.html">Malebranche, Nicolas</a> |
 <a href="../science-mechanisms/index.html">mechanism in science</a> |
 <a href="../metaphysics/index.html">metaphysics</a> |
 <a href="../mill/index.html">Mill, John Stuart</a> |
 <a href="../models-science/index.html">models in science</a> |
 <a href="../molecular-biology/index.html">molecular biology</a> |
 <a href="../reduction-biology/index.html">reduction, scientific: in biology</a> |
 <a href="../scientific-explanation/index.html">scientific explanation</a> |
 <a href="../systems-synthetic-biology/index.html">systems and synthetic biology, philosophy of</a> |
 <a href="../teleology-biology/index.html">teleology: teleological notions in biology</a>

</p>

</div>

<div id="acknowledgments">

<h3>Acknowledgments</h3>

<p>
Thanks to the many philosophical and scientific colleagues who have
provided me with extensive comments on different aspects of this
material over the past decade. I am grateful to Max Dresow, Kelle
Dhein, Nathan Lackey, Lauren Wilson, and Yoshinari Yoshida for
insightful recommendations and an anonymous referee for helpful
feedback that substantially improved the final version of the
entry.</p>
</div> 
</div><!-- #aueditable --><!--DO NOT MODIFY THIS LINE AND BELOW-->

<!-- END ARTICLE HTML -->

</div> <!-- End article-content -->

  <div id="article-copyright">
    <p>
 <a href="../../info.html#c">Copyright &copy; 2020</a> by

<br />
<a href="http://www.philosophy.umn.edu/people/FacultyProfile.php?UID=aclove" target="other">Alan Love</a>
&lt;<a href="m&#97;ilto:aclove&#37;40umn&#37;2eedu"><em>aclove<abbr title=" at ">&#64;</abbr>umn<abbr title=" dot ">&#46;</abbr>edu</em></a>&gt;
    </p>
  </div>

</div> <!-- End article -->

<!-- NOTE: article banner is outside of the id="article" div. -->
<div id="article-banner" class="scroll-block">
  <div id="article-banner-content">
    <a href="../../fundraising/index.html">
    Open access to the SEP is made possible by a world-wide funding initiative.<br />
    The Encyclopedia Now Needs Your Support<br />
    Please Read How You Can Help Keep the Encyclopedia Free</a>
  </div>
</div> <!-- End article-banner -->

    </div> <!-- End content -->

    <div id="footer">

      <div id="footer-menu">
        <div class="menu-block">
          <h4><i class="icon-book"></i> Browse</h4>
          <ul role="menu">
            <li><a href="../../contents.html">Table of Contents</a></li>
            <li><a href="../../new.html">What's New</a></li>
            <li><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/random">Random Entry</a></li>
            <li><a href="../../published.html">Chronological</a></li>
            <li><a href="../../archives/index.html">Archives</a></li>
          </ul>
        </div>
        <div class="menu-block">
          <h4><i class="icon-info-sign"></i> About</h4>
          <ul role="menu">
            <li><a href="../../info.html">Editorial Information</a></li>
            <li><a href="../../about.html">About the SEP</a></li>
            <li><a href="../../board.html">Editorial Board</a></li>
            <li><a href="../../cite.html">How to Cite the SEP</a></li>
            <li><a href="../../special-characters.html">Special Characters</a></li>
            <li><a href="../../tools/index.html">Advanced Tools</a></li>
            <li><a href="../../contact.html">Contact</a></li>
          </ul>
        </div>
        <div class="menu-block">
          <h4><i class="icon-leaf"></i> Support SEP</h4>
          <ul role="menu">
            <li><a href="../../support/index.html">Support the SEP</a></li>
            <li><a href="../../support/friends.html">PDFs for SEP Friends</a></li>
            <li><a href="../../support/donate.html">Make a Donation</a></li>
            <li><a href="../../support/sepia.html">SEPIA for Libraries</a></li>
          </ul>
        </div>
      </div> <!-- End footer menu -->

      <div id="mirrors">
        <div id="mirror-info">
          <h4><i class="icon-globe"></i> Mirror Sites</h4>
          <p>View this site from another server:</p>
        </div>
        <div class="btn-group open">
          <a class="btn dropdown-toggle" data-toggle="dropdown" href="https://plato.stanford.edu/">
            <span class="flag flag-usa"></span> USA (Main Site) <span class="caret"></span>
            <span class="mirror-source">Philosophy, Stanford University</span>
          </a>
          <ul class="dropdown-menu">
            <li><a href="../../mirrors.html">Info about mirror sites</a></li>
          </ul>
        </div>
      </div> <!-- End mirrors -->
      
      <div id="site-credits">
        <p>The Stanford Encyclopedia of Philosophy is <a href="../../info.html#c">copyright &copy; 2022</a> by <a href="http://mally.stanford.edu/">The Metaphysics Research Lab</a>, Department of Philosophy, Stanford University</p>
        <p>Library of Congress Catalog Data: ISSN 1095-5054</p>
      </div> <!-- End site credits -->

    </div> <!-- End footer -->

  </div> <!-- End container -->

   <!-- NOTE: Script required for drop-down button to work (mirrors). -->
  <script>
    $('.dropdown-toggle').dropdown();
  </script>

</body>

<!-- Mirrored from seop.illc.uva.nl/entries/biology-developmental/ by HTTrack Website Copier/3.x [XR&CO'2014], Wed, 22 Jun 2022 19:40:11 GMT -->
</html>
