<!DOCTYPE html>
<!--[if lt IE 7]> <html class="ie6 ie"> <![endif]-->
<!--[if IE 7]>    <html class="ie7 ie"> <![endif]-->
<!--[if IE 8]>    <html class="ie8 ie"> <![endif]-->
<!--[if IE 9]>    <html class="ie9 ie"> <![endif]-->
<!--[if !IE]> --> <html> <!-- <![endif]-->

<!-- Mirrored from seop.illc.uva.nl/entries/genomics/encode-project.html by HTTrack Website Copier/3.x [XR&CO'2014], Wed, 22 Jun 2022 20:12:15 GMT -->
<head>
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<title>
Genomics and Postgenomics &gt; The ENCODE Project and the ENCODE Controversy (Stanford Encyclopedia of Philosophy)
</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="robots" content="noarchive, noodp" />
<meta name="DCTERMS.ispartof" content="https://plato.stanford.edu/entries/genomics/" />

<!-- NOTE: Import webfonts using this link: -->
<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,300,600,200&amp;subset=latin,latin-ext" rel="stylesheet" type="text/css" />

<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/bootstrap.min.css" />
<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/bootstrap-responsive.min.css" />
<link rel="stylesheet" type="text/css" href="../../css/font-awesome.min.css" />
<!--[if IE 7]> <link rel="stylesheet" type="text/css" href="../../css/font-awesome-ie7.min.css"> <![endif]-->
<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/style.css" />
<link rel="stylesheet" type="text/css" media="print" href="../../css/print.css" />
<link rel="stylesheet" type="text/css" href="../../css/entry.css" />
<!--[if IE]> <link rel="stylesheet" type="text/css" href="../../css/ie.css" /> <![endif]-->
<script type="text/javascript" src="../../js/jquery-1.9.1.min.js"></script>
<script type="text/javascript" src="../../js/bootstrap.min.js"></script>

<!-- NOTE: Javascript for sticky behavior needed on article and ToC pages -->
<script type="text/javascript" src="../../js/jquery-scrolltofixed-min.js"></script>
<script type="text/javascript" src="../../js/entry.js"></script>

<!-- SEP custom script -->
<script type="text/javascript" src="../../js/sep.js"></script>
</head>

<!-- NOTE: The nojs class is removed from the page if javascript is enabled. Otherwise, it drives the display when there is no javascript. -->
<body class="nojs article" id="pagetopright">
<div id="container">
<div id="header-wrapper">
  <div id="header">
    <div id="branding">
      <div id="site-logo"><a href="../../index.html"><img src="../../symbols/sep-man-red.png" alt="SEP home page" /></a></div>
      <div id="site-title"><a href="../../index.html">Stanford Encyclopedia of Philosophy</a></div>
    </div>
    <div id="navigation">
      <div class="navbar">
        <div class="navbar-inner">
          <div class="container">
            <button class="btn btn-navbar collapsed" data-target=".collapse-main-menu" data-toggle="collapse" type="button"> <i class="icon-reorder"></i> Menu </button>
            <div class="nav-collapse collapse-main-menu in collapse">
              <ul class="nav">
                <li class="dropdown open"><a id="drop1" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-book"></i> Browse</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop1">
                    <li><a href="../../contents.html">Table of Contents</a></li>
                    <li><a href="../../new.html">What's New</a></li>
                    <li><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/random">Random Entry</a></li>
                    <li><a href="../../published.html">Chronological</a></li>
                    <li><a href="../../archives/index.html">Archives</a></li>
                  </ul>
                </li>
                <li class="dropdown open"><a id="drop2" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-info-sign"></i> About</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop2">
                    <li><a href="../../info.html">Editorial Information</a></li>
                    <li><a href="../../about.html">About the SEP</a></li>
                    <li><a href="../../board.html">Editorial Board</a></li>
                    <li><a href="../../cite.html">How to Cite the SEP</a></li>
                    <li><a href="../../special-characters.html">Special Characters</a></li>
                    <li><a href="../../tools/index.html">Advanced Tools</a></li>
                    <li><a href="../../contact.html">Contact</a></li>
                  </ul>
                </li>
                <li class="dropdown open"><a id="drop3" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-leaf"></i> Support SEP</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop3">
                    <li><a href="../../support/index.html">Support the SEP</a></li>
                    <li><a href="../../support/friends.html">PDFs for SEP Friends</a></li>
                    <li><a href="../../support/donate.html">Make a Donation</a></li>
                    <li><a href="../../support/sepia.html">SEPIA for Libraries</a></li>
                  </ul>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
    <!-- End navigation -->
    
    <div id="search">
      <form id="search-form" method="get" action="https://seop.illc.uva.nl/search/searcher.py">
        <input type="search" name="query" placeholder="Search SEP" />
        <div class="search-btn-wrapper"><button class="btn search-btn" type="submit"><i class="icon-search"></i></button></div>
      </form>
    </div>
    <!-- End search --> 
    
  </div>
  <!-- End header --> 
</div>
<!-- End header wrapper -->

<div id="content">

<!-- Begin article sidebar -->
<div id="article-sidebar" class="sticky">
  <div class="navbar">
    <div class="navbar-inner">
      <div class="container">
        <button class="btn btn-navbar" data-target=".collapse-sidebar" data-toggle="collapse" type="button"> <i class="icon-reorder"></i> Entry Navigation </button>
        <div id="article-nav" class="nav-collapse collapse-sidebar in collapse">
          <ul class="nav">
            <li><a href="index.html">Back to Entry <i class="icon-external-link"></i></a></li>
            <li><a href="index.html#toc">Entry Contents <i class="icon-external-link"></i></a></li>
            <li><a href="index.html#Bib">Entry Bibliography <i class="icon-external-link"></i></a></li>
            <li><a href="index.html#Aca">Academic Tools <i class="icon-external-link"></i></a></li>
            <li><a href="https://leibniz.stanford.edu/friends/preview/genomics/">Friends PDF Preview <i class="icon-external-link"></i></a></li>
            <li><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/archinfo.cgi?entry=genomics">Author and Citation Info <i class="icon-external-link"></i></a> </li>
            <li><a href="#pagetopright" class="back-to-top">Back to Top <i class="icon-angle-up icon2x"></i></a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
<!-- End article sidebar --> 

<!-- NOTE: Article content must have two wrapper divs: id="article" and id="article-content" -->
<div id="article">
<div id="article-content">

<!-- BEGIN ARTICLE HTML -->
<h4 id="supphead">Supplement to <a href="index-2.html">Genomics and Postgenomics</a></h4>

<div id="aueditable">
<!--DO NOT MODIFY THIS LINE AND ABOVE--> 
<h2>The ENCODE Project and the ENCODE Controversy</h2>

<p>
The ENCyclopedia Of DNA Elements (ENCODE) project was an international
research effort funded by the National Human Genome Research Institute
(NHGRI) that aimed to identify all functional elements (FE) in the
human genome (ENCODE Project Consortium 2004). FEs include, for
instance, protein-coding regions, regulatory elements such as
promoters, silencers or enhancers and sequences that are important for
chromosomal structure. The project, which began in 2003 and included
442 researchers during its main production phase, came to a conclusion
in 2012 with the publication of 30 different papers in different
journals (ENCODE Project Consortium 2012; Pennisi 2012). Similarly to
the HapMap project, ENCODE was presented as the logical next step
after the sequencing of the genomic DNA, since tackling the
<em>interpretation</em> of the sequences was now seen as the top
priority (ENCODE Project Consortium 2004).</p>

<p>
The ENCODE project incited a heated debate in academic journals, the
blogosphere and also in the national and international press. The
crucial claim that incited much ire was the project&rsquo;s conclusion
that 80.4% of the human genomic DNA has a &lsquo;biochemical
function&rsquo; (ENCODE Project Consortium 2012). To understand the
strong reaction this statement provoked we have to turn our focus
again to the C-value paradox and the concept of &lsquo;junk DNA&rsquo;
(see
 <a href="index-2.html#GenoSizeCValuParaJunkDNA">Section 2.3</a>
 of the main text). In the context of the ENCODE controversy this
debate was linked with the issue of how to define a &lsquo;functional
element&rsquo; and how scientists ascribe functions in biological
systems. What the ENCODE research implied, at least in the eyes of
some commentators, was that the idea of junk DNA was proven wrong,
because almost all of our DNA turned out to be functional. This led to
claims that textbooks will have to be re-written, as they still
describe the genome as mainly composed of
 junk.<sup>[<a href="notes.html#note-S1" id="ref-S1">S1</a>]</sup>
 The defenders of the old view claimed that the ENCODE researchers set
far too low a bar in ascribing functions to elements of biological
systems.</p>

<h3>The Methodology of the ENCODE Project</h3>

<p>
The ENCODE project used a range of different experimental assays to
analyse what they referred to as &lsquo;sites of biochemical
activity&rsquo; (for an overview of the ENCODE output see Qu &amp;
Fang 2013). These are sites at which some sort of modification can be
identified (for instance methylation) or to which an activity (such as
transcription of DNA to RNA) can be ascribed. These modifications or
activities were taken as strong indications that the identified
regions of the genomic DNA play a functional role in human cells.</p>

<p>
As an example of how this approach worked, ENCODE researchers were
interested in finding out how much of the genomic DNA is involved in
the regulation of gene expression. Researchers postulate that a key
hallmark of all regulatory DNA elements is their accessibility. This
makes sense as the regulatory and transcriptional machinery need
access to these DNA sites. ENCODE used this feature of regulatory DNA
to map (putative) regulatory elements in the human genome. One way to
do so is to perform what is called a &lsquo;DNase I hypersensitivity
assay&rsquo;. DNase I is a protein that can cut DNA and this cutting
process works better when the template DNA is accessible, meaning that
highly accessible regions are more sensitive to DNase I activity. The
behaviour of the genome in the DNase I hypersensitivity assay can
therefore be used to learn indirectly about its structure, from which
researchers then infer the presence of a functional element (in this
case a regulatory sequence). This is just one example of about 24
different types of assays that ENCODE researchers used to get a better
insight into the number and distribution of functional elements in the
human genome (for a discussion of the different types of experimental
approaches used in ENCODE see Kellis et al. 2014).</p>

<p>
What is interesting about most of these assays is that they look at a
<em>proxy</em> for function: if a stretch of DNA is hypersensitive to
DNase I then it is automatically defined as functional. Another
example is DNA transcription itself. If a DNA sequence shows up in RNA
sequencing then this means it has been transcribed into RNA by the
enzyme RNA polymerase. This activity, in the eyes of the ENCODE
researchers at least, makes the DNA element in question a functional
element of the genome.</p>

<p>
But such a broad approach to finding out about functional elements is
highly problematic, as a transcription event or hypersensitivity can
be present for many different reasons (for instance as a result of
transcriptional noise). This is exactly what some critics of ENCODE
homed in on, pointing out that merely showing the <em>existence</em>
of a structure (such as methylation) or a process (such as
transcription) is not enough by itself to prove any functional
significance of these biochemical features (Doolittle 2013; Eddy 2012;
Graur et al. 2013; Niu &amp; Jiang 2013).</p>

<p>
Whilst this is surely a valid point that applies to a large part of
the research done within ENCODE, not all studies performed as part of
the project looked at such proxies. An example is (Whitfield et al.
2012), who did not just look at specific modifications or behaviour of
DNA in particular assays but mutated specific sites to check whether
the interference with these sites has an effect on gene
expression.</p>

<p>
The above argument about how we learn about functional elements
presumes that we already have an understanding of what it means to be
&lsquo;functional&rsquo;. But it is by no means clear how biological
function should be defined and there are competing accounts of what it
is that makes an entity functional. These discussions about the
concept of a biological function were central to the dispute
surrounding the ENCODE project.</p>

<h3>The ENCODE Controversy</h3>

<p>
Especially in the early critiques by Doolittle (2013) and Graur et al.
(2013) the distinction between &lsquo;selected effect&rsquo; (SE)
function and the causal role (CR) function of an entity or process
figured prominently. The ENCODE project, so the critics, simply
ignored key work by philosophers and theoretical biologists on this
topic, thereby making a complete muddle of what they are talking about
when they use the term &lsquo;function&rsquo;. With more conceptual
clarity, they argued, the claim that 80% of our DNA is functional
would not be tenable and the established notion of &lsquo;junk
DNA&rsquo; would be saved.</p>

<p>
The definition of function and functional analysis in biology deserves
an SEP entry of its own. Here we will limit ourselves to a few
comments on this issue that relate to the ENCODE controversy
specifically. The key point is that SE functions are functions that
are assigned to <em>conserved</em> sequences. The SE account aims to
answer the question of why an element is there: a functional element
according to this definition is an entity whose presence has a
positive effect on the survival or reproduction of the organism;
meaning the entity has been selected for (Millikan 1984, 1989a;
Neander 1991; Griffiths 1992, 1993; Godfrey-Smith 1994). If a gene has
been selected for it is expected that its sequence will be conserved:
mutations within it will be selected against, and will be less
frequent than in sequences that are not being maintained by selection.
History matters for this account, which is also why it is sometimes
referred to as the etiological account of function, going back to a
paper by Wright (1976) but see Millikan (1989a) on how the etiological
account relates to Wright&rsquo;s original account).</p>

<p>
CR functions on the other hand do not depend on the history of the
system. What the CR account answers are &lsquo;how&rsquo; questions in
relation to the capacities of a system (Millikan 1989b). It is only
the here and now that matters for the CR account: functional analysis
is about analysing a system with capacity <i>C</i> into sub-capacities
that are attributed to elements of the system and which contribute to
<i>C</i> (Cummins 1975). The CR account is in an important sense more
liberal than the etiological account: according to CR anything can be
deemed a functional element as long as it is part of a system and
plays some causal role contributing to some system capacity we happen
to be interested in.</p>

<p>
Graur et al. (2013) claim that ENCODE worked with the CR account but
that this is a mistake, as biologists actually work with the SE
account, a claim that can also be found in Doolittle 2013. They
acknowledge that biologists might study CR functions (for instance
when doing deletion experiments) but claim that even if scientists do
so they take these causal roles simply as indicative of SE function,
which is the &lsquo;true&rsquo; function of a biological element
(Doolittle 2013; Graur et al. 2013). It is with this focus on SE
functions that these critics bring us back to the C-value paradox and
the strong case one can make for the importance of the junk DNA
concept.</p>

<p>
The deep problem is that there is simply not enough conserved DNA in
humans to match the high percentage of functional DNA the ENCODE
project came up with. Accepting current estimates that between 5 and
10% of the human genome is conserved (Lindblad-Toh et al. 2011; Ward
&amp; Kellis 2012) then there is clearly no correlation between the
amount of sequence under evolutionary constraint and what is called
&lsquo;functional&rsquo; by the ENCODE consortium. Graur et al. (2013)
call this the &lsquo;ENCODE
 incongruity&rsquo;.<sup>[<a href="notes.html#note-S2" id="ref-S2">S2</a>]</sup></p>
 
<p>
As already pointed out above this critique is based on a claim about
which functional account is actually used by scientists. This appears
to be taken as an empirical claim, though perhaps what matters more is
how scientists <em>ought</em> to understand functional language. This,
in turn, is likely to depend on what their aims are. Either way, this
is an important point, because once we think in terms of SE functions
DNA conservation immediately becomes salient. If, however, it turns
out that scientists don&rsquo;t (or shouldn&rsquo;t) use the SE
account (as is claimed, for instance, by Elliott et al. 2014; Germain
et al. 2014; Amundson &amp; Lauder 1994; Griffiths 1994, 2006) this
critique loses much of its force as the ENCODE incongruity is no
longer a problem.</p>

<p>
This is exactly the point on which a recently published critique of
the critics picks up. (Germain et al. 2014) claim that the critics of
ENCODE simply misunderstood the nature of the project, as they did not
take into account that the ENCODE project was part of a biomedical
discovery process. As such the project was concerned to find out about
elements of the human genome that might engage in <em>relevant
biochemical processes</em>. What makes a sequence or activity relevant
in the biomedical context it is not whether it is conserved but
whether its absence or presence has a potential effect on activities
or entities that are of relevance to biomedical research. The CR
account, Germain and co-workers claim, is therefore the right account
to use in this context and the &lsquo;ENCODE incongruity&rsquo; is no
longer a relevant issue.</p>

<p>
In all of this the ENCODE researchers themselves did not stay silent.
It is interesting to note that in a reply to their critics key ENCODE
members toned down their claims about the percentage of functional
elements present in the human genome &ndash; the 80.4% number is not
mentioned again (Kellis et al. 2014). In fact, no numbers are
mentioned in this paper and the authors remark that in their opinion
creating an open access resource (i.e., the ENCODE library) is
&ldquo;far more important than any interim estimate of the fraction of
the human genome that is functional&rdquo; (2014: 6136). The authors
also point out that in their eyes all experimental and theoretical
approaches to functional ascriptions have their limitations and that
no account or assay will get it right on their own, which is why they
advocate both a methodological and theoretical pluralism, again
defusing many of the stronger claims made earlier on both sides of the
dispute.</p>

<p>
The issue (Germain et al. 2014) raise concerns the <em>type</em> of
scientific project ENCODE is. As also in the context of the HapMap
project, we encounter the long-standing dispute about the value of
hypothesis-free or exploratory research (see
 <a href="index-2.html#HapMCrit">Section 3.1.3</a>
 of the main text). Eddy (2013), for instance, laments that the
project was originally a mapping project but was then spun
retrospectively as a project that aimed at testing a hypothesis. Graur
et al. (2013) also make the point that ENCODE overstepped their remit
of a big science project - which, they claim, is simply to provide
data &ndash; and that the ENCODE researchers ventured into
&lsquo;small science&rsquo; territory by trying to deliver an
interpretation of that data. In contrast to the criticisms the HGP
originally encountered, these modern-day critics don&rsquo;t have a
problem anymore with the <em>idea</em> of a descriptive mapping
project; their worry is rather that the project is sold as something
it isn&rsquo;t.</p> 


</div><!-- #aueditable --><!--DO NOT MODIFY THIS LINE AND BELOW-->

<!-- END ARTICLE HTML -->

</div> <!-- End article-content -->

  <div id="article-copyright">
    <p>
 <a href="../../info.html#c">Copyright &copy; 2016</a> by

<br />
Stephan Guttinger
&lt;<a href="m&#97;ilto:s&#37;2em&#37;2eguttinger&#37;40exeter&#37;2eac&#37;2euk"><em>s<abbr title=" dot ">&#46;</abbr>m<abbr title=" dot ">&#46;</abbr>guttinger<abbr title=" at ">&#64;</abbr>exeter<abbr title=" dot ">&#46;</abbr>ac<abbr title=" dot ">&#46;</abbr>uk</em></a>&gt;<br />
John Dupr&eacute;
&lt;<a href="m&#97;ilto:J&#37;2eA&#37;2eDupre&#37;40exeter&#37;2eac&#37;2euk"><em>J<abbr title=" dot ">&#46;</abbr>A<abbr title=" dot ">&#46;</abbr>Dupre<abbr title=" at ">&#64;</abbr>exeter<abbr title=" dot ">&#46;</abbr>ac<abbr title=" dot ">&#46;</abbr>uk</em></a>&gt;
    </p>
  </div>

</div> <!-- End article -->

<!-- NOTE: article banner is outside of the id="article" div. -->
<div id="article-banner" class="scroll-block">
  <div id="article-banner-content">
    <a href="../../fundraising/index.html">
    Open access to the SEP is made possible by a world-wide funding initiative.<br />
    The Encyclopedia Now Needs Your Support<br />
    Please Read How You Can Help Keep the Encyclopedia Free</a>
  </div>
</div> <!-- End article-banner -->

    </div> <!-- End content -->

    <div id="footer">

      <div id="footer-menu">
        <div class="menu-block">
          <h4><i class="icon-book"></i> Browse</h4>
          <ul role="menu">
            <li><a href="../../contents.html">Table of Contents</a></li>
            <li><a href="../../new.html">What's New</a></li>
            <li><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/random">Random Entry</a></li>
            <li><a href="../../published.html">Chronological</a></li>
            <li><a href="../../archives/index.html">Archives</a></li>
          </ul>
        </div>
        <div class="menu-block">
          <h4><i class="icon-info-sign"></i> About</h4>
          <ul role="menu">
            <li><a href="../../info.html">Editorial Information</a></li>
            <li><a href="../../about.html">About the SEP</a></li>
            <li><a href="../../board.html">Editorial Board</a></li>
            <li><a href="../../cite.html">How to Cite the SEP</a></li>
            <li><a href="../../special-characters.html">Special Characters</a></li>
            <li><a href="../../tools/index.html">Advanced Tools</a></li>
            <li><a href="../../contact.html">Contact</a></li>
          </ul>
        </div>
        <div class="menu-block">
          <h4><i class="icon-leaf"></i> Support SEP</h4>
          <ul role="menu">
            <li><a href="../../support/index.html">Support the SEP</a></li>
            <li><a href="../../support/friends.html">PDFs for SEP Friends</a></li>
            <li><a href="../../support/donate.html">Make a Donation</a></li>
            <li><a href="../../support/sepia.html">SEPIA for Libraries</a></li>
          </ul>
        </div>
      </div> <!-- End footer menu -->

      <div id="mirrors">
        <div id="mirror-info">
          <h4><i class="icon-globe"></i> Mirror Sites</h4>
          <p>View this site from another server:</p>
        </div>
        <div class="btn-group open">
          <a class="btn dropdown-toggle" data-toggle="dropdown" href="https://plato.stanford.edu/">
            <span class="flag flag-usa"></span> USA (Main Site) <span class="caret"></span>
            <span class="mirror-source">Philosophy, Stanford University</span>
          </a>
          <ul class="dropdown-menu">
            <li><a href="../../mirrors.html">Info about mirror sites</a></li>
          </ul>
        </div>
      </div> <!-- End mirrors -->
      
      <div id="site-credits">
        <p>The Stanford Encyclopedia of Philosophy is <a href="../../info.html#c">copyright &copy; 2022</a> by <a href="http://mally.stanford.edu/">The Metaphysics Research Lab</a>, Department of Philosophy, Stanford University</p>
        <p>Library of Congress Catalog Data: ISSN 1095-5054</p>
      </div> <!-- End site credits -->

    </div> <!-- End footer -->

  </div> <!-- End container -->

   <!-- NOTE: Script required for drop-down button to work (mirrors). -->
  <script>
    $('.dropdown-toggle').dropdown();
  </script>

</body>

<!-- Mirrored from seop.illc.uva.nl/entries/genomics/encode-project.html by HTTrack Website Copier/3.x [XR&CO'2014], Wed, 22 Jun 2022 20:12:15 GMT -->
</html>
