<!DOCTYPE html>
<!--[if lt IE 7]> <html class="ie6 ie"> <![endif]-->
<!--[if IE 7]>    <html class="ie7 ie"> <![endif]-->
<!--[if IE 8]>    <html class="ie8 ie"> <![endif]-->
<!--[if IE 9]>    <html class="ie9 ie"> <![endif]-->
<!--[if !IE]> --> <html> <!-- <![endif]-->

<!-- Mirrored from seop.illc.uva.nl/entries/genomics/notes.html by HTTrack Website Copier/3.x [XR&CO'2014], Wed, 22 Jun 2022 20:12:14 GMT -->
<head>
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<title>
Genomics and Postgenomics &gt; Notes (Stanford Encyclopedia of Philosophy)
</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="robots" content="noarchive, noodp" />
<meta name="DCTERMS.ispartof" content="https://plato.stanford.edu/entries/genomics/" />

<!-- NOTE: Import webfonts using this link: -->
<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,300,600,200&amp;subset=latin,latin-ext" rel="stylesheet" type="text/css" />

<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/bootstrap.min.css" />
<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/bootstrap-responsive.min.css" />
<link rel="stylesheet" type="text/css" href="../../css/font-awesome.min.css" />
<!--[if IE 7]> <link rel="stylesheet" type="text/css" href="../../css/font-awesome-ie7.min.css"> <![endif]-->
<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/style.css" />
<link rel="stylesheet" type="text/css" media="print" href="../../css/print.css" />
<link rel="stylesheet" type="text/css" href="../../css/entry.css" />
<!--[if IE]> <link rel="stylesheet" type="text/css" href="../../css/ie.css" /> <![endif]-->
<script type="text/javascript" src="../../js/jquery-1.9.1.min.js"></script>
<script type="text/javascript" src="../../js/bootstrap.min.js"></script>

<!-- NOTE: Javascript for sticky behavior needed on article and ToC pages -->
<script type="text/javascript" src="../../js/jquery-scrolltofixed-min.js"></script>
<script type="text/javascript" src="../../js/entry.js"></script>

<!-- SEP custom script -->
<script type="text/javascript" src="../../js/sep.js"></script>
</head>

<!-- NOTE: The nojs class is removed from the page if javascript is enabled. Otherwise, it drives the display when there is no javascript. -->
<body class="nojs article" id="pagetopright">
<div id="container">
<div id="header-wrapper">
  <div id="header">
    <div id="branding">
      <div id="site-logo"><a href="../../index.html"><img src="../../symbols/sep-man-red.png" alt="SEP home page" /></a></div>
      <div id="site-title"><a href="../../index.html">Stanford Encyclopedia of Philosophy</a></div>
    </div>
    <div id="navigation">
      <div class="navbar">
        <div class="navbar-inner">
          <div class="container">
            <button class="btn btn-navbar collapsed" data-target=".collapse-main-menu" data-toggle="collapse" type="button"> <i class="icon-reorder"></i> Menu </button>
            <div class="nav-collapse collapse-main-menu in collapse">
              <ul class="nav">
                <li class="dropdown open"><a id="drop1" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-book"></i> Browse</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop1">
                    <li><a href="../../contents.html">Table of Contents</a></li>
                    <li><a href="../../new.html">What's New</a></li>
                    <li><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/random">Random Entry</a></li>
                    <li><a href="../../published.html">Chronological</a></li>
                    <li><a href="../../archives/index.html">Archives</a></li>
                  </ul>
                </li>
                <li class="dropdown open"><a id="drop2" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-info-sign"></i> About</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop2">
                    <li><a href="../../info.html">Editorial Information</a></li>
                    <li><a href="../../about.html">About the SEP</a></li>
                    <li><a href="../../board.html">Editorial Board</a></li>
                    <li><a href="../../cite.html">How to Cite the SEP</a></li>
                    <li><a href="../../special-characters.html">Special Characters</a></li>
                    <li><a href="../../tools/index.html">Advanced Tools</a></li>
                    <li><a href="../../contact.html">Contact</a></li>
                  </ul>
                </li>
                <li class="dropdown open"><a id="drop3" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-leaf"></i> Support SEP</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop3">
                    <li><a href="../../support/index.html">Support the SEP</a></li>
                    <li><a href="../../support/friends.html">PDFs for SEP Friends</a></li>
                    <li><a href="../../support/donate.html">Make a Donation</a></li>
                    <li><a href="../../support/sepia.html">SEPIA for Libraries</a></li>
                  </ul>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
    <!-- End navigation -->
    
    <div id="search">
      <form id="search-form" method="get" action="https://seop.illc.uva.nl/search/searcher.py">
        <input type="search" name="query" placeholder="Search SEP" />
        <div class="search-btn-wrapper"><button class="btn search-btn" type="submit"><i class="icon-search"></i></button></div>
      </form>
    </div>
    <!-- End search --> 
    
  </div>
  <!-- End header --> 
</div>
<!-- End header wrapper -->

<div id="content">

<!-- Begin article sidebar -->
<div id="article-sidebar" class="sticky">
  <div class="navbar">
    <div class="navbar-inner">
      <div class="container">
        <button class="btn btn-navbar" data-target=".collapse-sidebar" data-toggle="collapse" type="button"> <i class="icon-reorder"></i> Entry Navigation </button>
        <div id="article-nav" class="nav-collapse collapse-sidebar in collapse">
          <ul class="nav">
            <li><a href="index.html">Back to Entry <i class="icon-external-link"></i></a></li>
            <li><a href="index.html#toc">Entry Contents <i class="icon-external-link"></i></a></li>
            <li><a href="index.html#Bib">Entry Bibliography <i class="icon-external-link"></i></a></li>
            <li><a href="index.html#Aca">Academic Tools <i class="icon-external-link"></i></a></li>
            <li><a href="https://leibniz.stanford.edu/friends/preview/genomics/">Friends PDF Preview <i class="icon-external-link"></i></a></li>
            <li><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/archinfo.cgi?entry=genomics">Author and Citation Info <i class="icon-external-link"></i></a> </li>
            <li><a href="#pagetopright" class="back-to-top">Back to Top <i class="icon-angle-up icon2x"></i></a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
<!-- End article sidebar --> 

<!-- NOTE: Article content must have two wrapper divs: id="article" and id="article-content" -->
<div id="article">
<div id="article-content">

<!-- BEGIN ARTICLE HTML -->
<h4 id="supphead"></h4>

<div id="aueditable">
<!--DO NOT MODIFY THIS LINE AND ABOVE--> 
<h2>Notes to <a href="index-2.html">Genomics and Postgenomics</a></h2>

<p>

 <a href="index-2.html#ref-1" id="note-1">1.</a>
 DNA methylation refers to the process in which a methyl group
(-CH<sub>3</sub>) is covalently added to another molecule (in this
case DNA). The importance of this process will be discussed further
below.</p>

<p>

 <a href="index-2.html#ref-2" id="note-2">2.</a>
 A minor problem that we shall not discuss is that some viruses do not
contain DNA at all, but use RNA to serve their basic genetic
functions. This is a fairly technical problem and not too hard to
resolve with a qualification to the definition. We should note,
however, that it is not a problem at all for the account of the genome
that we favour (below).</p>

<p>

 <a href="index-2.html#ref-3" id="note-3">3.</a>
 Though perhaps this is not an indefensible move. A more radical way
to approach this issue is to abandon the idea of &lsquo;the&rsquo;
genome of an organism and treat both the organism and its genome as
complex symbiotic wholes, &ldquo;holobionts&rdquo;, polygenomic
entities with multiple and diverse genomes (Dupr&eacute; 2010;
reprinted as Dupr&eacute; 2012: ch. 7).</p>

<p>

 <a href="index-2.html#ref-4" id="note-4">4.</a>
 Non-coding DNA is defined as DNA that does not code for
<em>proteins</em>. It is estimated that only about 1.5% of human
chromosomal DNA codes for proteins (International Human Genome
Sequencing Consortium 2001, 2004). The ENCODE project (see the 
 <a href="encode-project.html">Supplement</a>)
 and other recent research on DNA transcription has shown that a large
part of chromosomal DNA (up to 74%) is transcribed into RNA, the
majority of which does not code for proteins (which is why it is also
called non-coding RNA (ncRNA)), but a substantial though debated
proportion of which is known to serve some function. For an overview
of types of ncRNA see Wright and Bruford 2011. It is estimated by some
that there could be as many or more genes for non-coding RNA than for
coding RNA, bringing up the gene count from 20,000 to at least 40,000
or more (Harrow et al. 2012; Clark et al. 2013).</p>

<p>

 <a href="index-2.html#ref-5" id="note-5">5.</a>
 Keller (2011) identifies four different definitions of the term:
&lsquo;genome-1&rsquo; is defined as the set of genes,
&lsquo;genome-2&rsquo; as the set of chromosomes in a cell.
&lsquo;Genome-3&rsquo; is defined as the organism&rsquo;s DNA, whereas
&lsquo;genome-4&rsquo; refers to all the genetic material of an
organism.</p>

<p>

 <a href="index-2.html#ref-6" id="note-6">6.</a>
 For further discussion of the broader processual view of biology of
which this view of the genome is a vital part, see Dupr&eacute; 2012;
Nicholson and Dupr&eacute; forthcoming.</p>

<p>

 <a href="index-2.html#ref-7" id="note-7">7.</a>
 We shall not address here the fascinating history of sequencing
technologies, from the laborious manual techniques of the 70s and 80s,
to the machines that today can provide gigabytes of sequence data in a
few hours. Note that as crucial as speed is accuracy, and in actual
sequencing practice a large number of runs is required to reduce
errors.</p>

<p>

 <a href="index-2.html#ref-8" id="note-8">8.</a>
 These discussions about the different modes of the experimental
sciences (exploratory vs. hypothesis- or theory-driven
experimentation) also became an important topic in philosophy of
science (see, e.g., Steinle 1997; Burian 1997; Franklin 2005; Elliott
2007; O&rsquo;Malley 2007; Waters 2007a; Karaca 2013).
Philosophers of science also tackled the important issue of how these
different modes of research are integrated with each other (Burian
2007; O&rsquo;Malley et al. 2010; O&rsquo;Malley &amp; Soyer
2012).</p>

<p>

 <a href="index-2.html#ref-9" id="note-9">9.</a>
 Fittingly, the first genome of a <em>single</em> person to be
published was that of James D. Watson himself, whose sequence was
released online in 2007 and published in an academic journal in 2008
(Wheeler et al. 2008). The online release of Watson&rsquo;s genome was
quickly followed by the publication of Craig Venter&rsquo;s genome
(Levy et al. 2007).</p>

<p>

 <a href="index-2.html#ref-10" id="note-10">10.</a>  By enlarging the
range of sequenced genomes the HGP also had a crucial impact on what
is now called &lsquo;comparative genomics&rsquo;. The power of this
discipline lies in the insights it can give us into the relations
between different organisms or species at the genetic level (Touchman
2010). The comparison can take place at different resolutions (overall
genome size, number of chromosomes, genes or, most importantly
perhaps, nucleotide-by-nucleotide alignments of different
sequences). The field has been of crucial importance to phylogenetics
and has also attracted the attention of philosophers (see, e.g., Moss
2006; Piotrowska 2009; Perini 2011). Even though, due to space
restrictions, we could not include a section dedicated to comparative
genomics, we will discuss crucial aspects of this practice throughout
this entry, as its tools and methods play a key role in many of the
projects we touch upon (such as the HapMap discussed in Section 3.1 or
the ENCODE project discussed in a supplement).</p>
 
<p>

 <a href="index-2.html#ref-11" id="note-11">11.</a>
 Note that we can find the term &lsquo;junk DNA&rsquo; earlier, for
instance in a paper by Ehret and De Haller (1963), where the authors
mention that </p>

<blockquote>

<p>
[w]hile current evidence makes plausible the idea that all genetic
material is DNA (with the possible exception of RNA viruses), it does
not follow that all DNA is competent genetic material (viz.
&ldquo;junk&rdquo; DNA) [&hellip;]. (Ehret &amp; De Haller 1963: 39)
</p>
</blockquote>

<p>
See Graur 2013, for more on the history of the term &lsquo;junk
DNA&rsquo;.</p>

<p>

 <a href="index-2.html#ref-12" id="note-12">12.</a>
 Note how assumption (a) neglects the key examples that gave rise to
the C-value paradox.</p>

<p>

 <a href="index-2.html#ref-13" id="note-13">13.</a>
 As Evelyn Fox Keller (2011) points out, this concept of junk DNA
became a staple of genomic thinking in the 1980s, in particular
through the publication of two papers (Doolittle &amp; Sapienza 1980;
Orgel &amp; Crick 1980) that linked Richard Dawkins&rsquo; (1976)
concept of selfish DNA to Ohno&rsquo;s notion of junk DNA.</p>

<p>

<a href="index-2.html#ref-14" id="note-14">14.</a> The official NCBI
website containing the HapMap resource data was decommissioned in June
2016 due to a security issue that was uncovered in a computer security
audit (see the link to &lsquo;Decomissioned NCBI HapMap
Resource&rsquo; in the Other Internet Resources section). The NCBI
justifies the decision to decommission the resource by a decline in
usage that could be observed over recent years. This has sparked a
conversation amongst scientists who believe in the continuing
importance of the HapMap resource. Parts of these discussions can be
found on Twitter under the hashtag #saveHapMap.</p>

<p>

 <a href="index-2.html#ref-15" id="note-15">15.</a>
 The term &lsquo;allele&rsquo; is often used to refer to different
variants of a gene but the term is now used more liberally and can
simply refer to variants of any locus on chromosomal DNA. See for
instance talking glossary: allele or scitable: allele under OIR.</p>

<p>

 <a href="index-2.html#ref-16" id="note-16">16.</a>
 Note that 1% is an arbitrary value. Some authors, for instance, set
the threshold at 5%.</p>

<p>

 <a href="index-2.html#ref-17" id="note-17">17.</a>
 Meiosis is the special type of cell division that happens only in an
organism&rsquo;s reproductive cells (gametes). During meiosis
homologous chromosomes (i.e., the maternal and paternal version of
each chromosome) become fragmented and are then rejoined in a process
called &lsquo;crossing-over&rsquo;. This process leads to the
(homologous) recombination of parts of the two chromosomes. If a
crossing over happens between two different loci on a chromosome the
alleles at these two loci become separated. If the loci are closer to
each other then the likelihood of a crossing over event taking place
between them is smaller, leading to a higher association level between
the two alleles.</p>

<p>

 <a href="index-2.html#ref-18" id="note-18">18.</a>
 For a short history of the CD/CV hypothesis see Box 2 in Visscher et
al. 2012.</p>

<p>

 <a href="index-2.html#ref-19" id="note-19">19.</a>
 Francis Collins was quoted as saying that the HapMap will be
&ldquo;the single most important genomic resource for understanding
human disease, after the sequence&rdquo; (Couzin 2002).</p>

<p>

 <a href="index-2.html#ref-20" id="note-20">20.</a>
 Hall (2010) also cites other prominent researchers, such as Walter
Bodmer or David Goldstein, who were critical of the HapMap
project.</p>

<p>

 <a href="index-2.html#ref-21" id="note-21">21.</a>
 So US president Bill Clinton talking on June 26, 2000 at the
presentation of the draft genome at the White House: </p>

<blockquote>

<p>
With this profound new knowledge, humankind is on the verge of gaining
immense, new power to heal. Genome science will have a real impact on
all our lives&mdash;and even more, on the lives of our children. It
will revolutionize the diagnosis, prevention and treatment of most, if
not all, human diseases. (White House 2000)</p>
</blockquote>

<p>

 <a href="index-2.html#ref-22" id="note-22">22.</a>
 For an overview and discussion of the literature on race (and ethnic
categories) as socially and historically constructed see Fujimura et
al. 2014: 209. Lisa Gannett claims that the concept of race has never
really been abandoned in biology but was rather transformed from a
typological into a population-based understanding of race (Gannett
2001). For more on the history of the race concept see Stolley (1999),
Marks (2008), Yudell (2011), and the SEP entry on
 <a href="../race/index.html">race</a>.</p>
 
<p>

 <a href="index-2.html#ref-23" id="note-23">23.</a>
 The anomaly consists of the discrepancy between the proportion of
living microbes that can be detected in the original sample using a
microscope and the proportion of microbes of that same sample that
then grow on an agar plate, the key means of culturing microbes.</p>

<p>

 <a href="index-2.html#ref-24" id="note-24">24.</a>
 This step is usually performed in the bacterium <em>E. coli</em>. See
Streit and Schmitz 2004, National Research Council 2007, and Ekkers et
al. 2012 for a discussion of the methodological difficulties such an
approach can encounter.</p>

<h2>Notes to the Supplement</h2>

<p>

 <a href="encode-project.html#ref-S1" id="note-S1">S1.</a>
 Interestingly, the term &lsquo;junk DNA&rsquo; does not appear in any
of the main consortium publications and there are no claims made about
the &lsquo;death&rsquo; of the junk DNA concept (see ENCODE Project
Consortium 2007, 2012). But as Germain et al. (2014) point out, we
find the term in commentaries such as Ecker et al. 2012, Pennisi 2012
with Pennisi in particular claiming that the ENCODE project has
written a &lsquo;eulogy&rsquo; for junk DNA. It is also these
editorials (or articles written in newspapers such as the New York
Times) to which Doolittle (2013) turned in his critique of the ENCODE
project and its claims about the demise of the junk DNA concept. But
see Eddy (2013) for the claim that some of the ENCODE leaders
themselves spun the project towards the textbooks-are-wrong narrative
in their attempts to popularize the project.</p>

<p>

 <a href="encode-project.html#ref-S2" id="note-S2">S2.</a>
 It has to be noted here that this is in fact something the ENCODE
researchers already noted in the publication reporting on the pilot
phase of the project (ENCODE Project Consortium 2007). The fact that
their assays picked up so much more biochemical activity than what
would be expected if we only look at conserved regions of the genome
was something they repeatedly highlighted and also tried to explain
(ENCODE Project Consortium 2007).</p> 



</div><!-- #aueditable --><!--DO NOT MODIFY THIS LINE AND BELOW-->

<!-- END ARTICLE HTML -->

</div> <!-- End article-content -->

  <div id="article-copyright">
    <p>
 <a href="../../info.html#c">Copyright &copy; 2016</a> by

<br />
Stephan Guttinger
&lt;<a href="m&#97;ilto:s&#37;2em&#37;2eguttinger&#37;40exeter&#37;2eac&#37;2euk"><em>s<abbr title=" dot ">&#46;</abbr>m<abbr title=" dot ">&#46;</abbr>guttinger<abbr title=" at ">&#64;</abbr>exeter<abbr title=" dot ">&#46;</abbr>ac<abbr title=" dot ">&#46;</abbr>uk</em></a>&gt;<br />
John Dupr&eacute;
&lt;<a href="m&#97;ilto:J&#37;2eA&#37;2eDupre&#37;40exeter&#37;2eac&#37;2euk"><em>J<abbr title=" dot ">&#46;</abbr>A<abbr title=" dot ">&#46;</abbr>Dupre<abbr title=" at ">&#64;</abbr>exeter<abbr title=" dot ">&#46;</abbr>ac<abbr title=" dot ">&#46;</abbr>uk</em></a>&gt;
    </p>
  </div>

</div> <!-- End article -->

<!-- NOTE: article banner is outside of the id="article" div. -->
<div id="article-banner" class="scroll-block">
  <div id="article-banner-content">
    <a href="../../fundraising/index.html">
    Open access to the SEP is made possible by a world-wide funding initiative.<br />
    The Encyclopedia Now Needs Your Support<br />
    Please Read How You Can Help Keep the Encyclopedia Free</a>
  </div>
</div> <!-- End article-banner -->

    </div> <!-- End content -->

    <div id="footer">

      <div id="footer-menu">
        <div class="menu-block">
          <h4><i class="icon-book"></i> Browse</h4>
          <ul role="menu">
            <li><a href="../../contents.html">Table of Contents</a></li>
            <li><a href="../../new.html">What's New</a></li>
            <li><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/random">Random Entry</a></li>
            <li><a href="../../published.html">Chronological</a></li>
            <li><a href="../../archives/index.html">Archives</a></li>
          </ul>
        </div>
        <div class="menu-block">
          <h4><i class="icon-info-sign"></i> About</h4>
          <ul role="menu">
            <li><a href="../../info.html">Editorial Information</a></li>
            <li><a href="../../about.html">About the SEP</a></li>
            <li><a href="../../board.html">Editorial Board</a></li>
            <li><a href="../../cite.html">How to Cite the SEP</a></li>
            <li><a href="../../special-characters.html">Special Characters</a></li>
            <li><a href="../../tools/index.html">Advanced Tools</a></li>
            <li><a href="../../contact.html">Contact</a></li>
          </ul>
        </div>
        <div class="menu-block">
          <h4><i class="icon-leaf"></i> Support SEP</h4>
          <ul role="menu">
            <li><a href="../../support/index.html">Support the SEP</a></li>
            <li><a href="../../support/friends.html">PDFs for SEP Friends</a></li>
            <li><a href="../../support/donate.html">Make a Donation</a></li>
            <li><a href="../../support/sepia.html">SEPIA for Libraries</a></li>
          </ul>
        </div>
      </div> <!-- End footer menu -->

      <div id="mirrors">
        <div id="mirror-info">
          <h4><i class="icon-globe"></i> Mirror Sites</h4>
          <p>View this site from another server:</p>
        </div>
        <div class="btn-group open">
          <a class="btn dropdown-toggle" data-toggle="dropdown" href="https://plato.stanford.edu/">
            <span class="flag flag-usa"></span> USA (Main Site) <span class="caret"></span>
            <span class="mirror-source">Philosophy, Stanford University</span>
          </a>
          <ul class="dropdown-menu">
            <li><a href="../../mirrors.html">Info about mirror sites</a></li>
          </ul>
        </div>
      </div> <!-- End mirrors -->
      
      <div id="site-credits">
        <p>The Stanford Encyclopedia of Philosophy is <a href="../../info.html#c">copyright &copy; 2022</a> by <a href="http://mally.stanford.edu/">The Metaphysics Research Lab</a>, Department of Philosophy, Stanford University</p>
        <p>Library of Congress Catalog Data: ISSN 1095-5054</p>
      </div> <!-- End site credits -->

    </div> <!-- End footer -->

  </div> <!-- End container -->

   <!-- NOTE: Script required for drop-down button to work (mirrors). -->
  <script>
    $('.dropdown-toggle').dropdown();
  </script>

</body>

<!-- Mirrored from seop.illc.uva.nl/entries/genomics/notes.html by HTTrack Website Copier/3.x [XR&CO'2014], Wed, 22 Jun 2022 20:12:15 GMT -->
</html>
